{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# **APPROACH:** The newest implementation outlined:\n",
    "- open the meta CSV file in a dataframe - 44K entries\n",
    "- drop the duplicates by abstract\n",
    "- loop (comprehension) a list of lists for the keyword for searches. eg. search=[['incubation','period','range']]\n",
    "- stem the keywords so incubation becomes incubat etc.\n",
    "- append indpendently - different variations of corona references - (covid, cov, -cov, hcov) to the keywords avoids pulling older research\n",
    "- query function in pandas to query the above terms with the keywords - the abstract has to have all the keywords and at least one of (covid, cov, -cov, hcov) \n",
    "- drop the duplicates in the df again after the queries\n",
    "- caclulate a relevance score for all the abstracts in the query result df\n",
    "- raw_score = total count of the keywords in the abstract\n",
    "- final_score = (raw_score/len(abstract))*raw_score\n",
    "- sort the df on the final_score ascending=False (best on top)\n",
    "- drop rows with a relevance score < .02\n",
    "- parse the relevant abstarcts into sentences split('. ') works well.\n",
    "- test if keywords are in the sentences\n",
    "- if sentence has all keywords, it is added to the df for display.\n",
    "- if you are seeking statistics in the data, adding % to the search terms works well\n",
    "- df with search results displayed in HTML - currently limiting results to 3 for ease of scanning\n",
    "\n",
    "**Pros:** Currently the system is a very simple **(as Einstein said \"make it as simple as possible, but no simpler\")**, but quite effective solutiuon to providing insight on topical queries.  The code is easy to read and simple to understand.\n",
    "\n",
    "**Cons:** Currently the system requires some human understanding in crafting keyword combinations, however, the recent relevance measure changes and stemming of keywords, have made it so the keywords can be very close to the NL questions and return very good results.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np # linear algebra\n",
    "import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)\n",
    "\n",
    "# keep only documents with covid -cov-2 and cov2\n",
    "def search_focus(df):\n",
    "    dfa = df[df['abstract'].str.contains('covid')]\n",
    "    dfb = df[df['abstract'].str.contains('-cov-2')]\n",
    "    dfc = df[df['abstract'].str.contains('cov2')]\n",
    "    dfd = df[df['abstract'].str.contains('ncov')]\n",
    "    frames=[dfa,dfb,dfc,dfd]\n",
    "    df = pd.concat(frames)\n",
    "    df=df.drop_duplicates(subset='title', keep=\"first\")\n",
    "    return df\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3063: DtypeWarning: Columns (1,4) have mixed types.Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(204823, 7)\n",
      "(66159, 7)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>sha</th>\n",
       "      <th>title</th>\n",
       "      <th>doi</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "      <th>journal</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>4662</th>\n",
       "      <td>no data provided</td>\n",
       "      <td>Latest assessment on COVID-19 from the Europea...</td>\n",
       "      <td>10.2807/1560-7917.es.2020.25.8.2002271</td>\n",
       "      <td>no data providedlatest assessment on covid-19 ...</td>\n",
       "      <td>2020-02-27</td>\n",
       "      <td>no data provided</td>\n",
       "      <td>Euro Surveill</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4698</th>\n",
       "      <td>no data provided</td>\n",
       "      <td>Updated rapid risk assessment from ECDC on the...</td>\n",
       "      <td>10.2807/1560-7917.es.2020.25.9.2003051</td>\n",
       "      <td>no data providedupdated rapid risk assessment ...</td>\n",
       "      <td>2020-03-05</td>\n",
       "      <td>no data provided</td>\n",
       "      <td>Euro Surveill</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4732</th>\n",
       "      <td>no data provided</td>\n",
       "      <td>Updated rapid risk assessment from ECDC on the...</td>\n",
       "      <td>10.2807/1560-7917.es.2020.25.10.2003121</td>\n",
       "      <td>no data providedupdated rapid risk assessment ...</td>\n",
       "      <td>2020-03-12</td>\n",
       "      <td>no data provided</td>\n",
       "      <td>Euro Surveill</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4800</th>\n",
       "      <td>601e6ac1ad98e359dc021e8896a1a604331ca774</td>\n",
       "      <td>Empfehlungen zur intensivmedizinischen Therapi...</td>\n",
       "      <td>10.1007/s00063-020-00674-3</td>\n",
       "      <td>no data providedempfehlungen zur intensivmediz...</td>\n",
       "      <td>2020-03-12</td>\n",
       "      <td>Kluge, Stefan; Janssens, Uwe; Welte, Tobias; W...</td>\n",
       "      <td>Med Klin Intensivmed Notfmed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5683</th>\n",
       "      <td>no data provided</td>\n",
       "      <td>The impact of COVID-19 on the provision of don...</td>\n",
       "      <td>10.1038/s41409-020-0873-x</td>\n",
       "      <td>no data providedthe impact of covid-19 on the ...</td>\n",
       "      <td>2020-03-23</td>\n",
       "      <td>Szer, Jeff; Weisdorf, Daniel; Querol, Sergio; ...</td>\n",
       "      <td>Bone Marrow Transplant</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                           sha  \\\n",
       "4662                          no data provided   \n",
       "4698                          no data provided   \n",
       "4732                          no data provided   \n",
       "4800  601e6ac1ad98e359dc021e8896a1a604331ca774   \n",
       "5683                          no data provided   \n",
       "\n",
       "                                                  title  \\\n",
       "4662  Latest assessment on COVID-19 from the Europea...   \n",
       "4698  Updated rapid risk assessment from ECDC on the...   \n",
       "4732  Updated rapid risk assessment from ECDC on the...   \n",
       "4800  Empfehlungen zur intensivmedizinischen Therapi...   \n",
       "5683  The impact of COVID-19 on the provision of don...   \n",
       "\n",
       "                                          doi  \\\n",
       "4662   10.2807/1560-7917.es.2020.25.8.2002271   \n",
       "4698   10.2807/1560-7917.es.2020.25.9.2003051   \n",
       "4732  10.2807/1560-7917.es.2020.25.10.2003121   \n",
       "4800               10.1007/s00063-020-00674-3   \n",
       "5683                10.1038/s41409-020-0873-x   \n",
       "\n",
       "                                               abstract publish_time  \\\n",
       "4662  no data providedlatest assessment on covid-19 ...   2020-02-27   \n",
       "4698  no data providedupdated rapid risk assessment ...   2020-03-05   \n",
       "4732  no data providedupdated rapid risk assessment ...   2020-03-12   \n",
       "4800  no data providedempfehlungen zur intensivmediz...   2020-03-12   \n",
       "5683  no data providedthe impact of covid-19 on the ...   2020-03-23   \n",
       "\n",
       "                                                authors  \\\n",
       "4662                                   no data provided   \n",
       "4698                                   no data provided   \n",
       "4732                                   no data provided   \n",
       "4800  Kluge, Stefan; Janssens, Uwe; Welte, Tobias; W...   \n",
       "5683  Szer, Jeff; Weisdorf, Daniel; Querol, Sergio; ...   \n",
       "\n",
       "                           journal  \n",
       "4662                 Euro Surveill  \n",
       "4698                 Euro Surveill  \n",
       "4732                 Euro Surveill  \n",
       "4800  Med Klin Intensivmed Notfmed  \n",
       "5683        Bone Marrow Transplant  "
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load the meta data from the CSV file using 3 columns (abstract, title, authors),\n",
    "df=pd.read_csv('metadata.csv', usecols=['title','journal','abstract','authors','doi','publish_time','sha'])\n",
    "print (df.shape)\n",
    "#drop duplicates\n",
    "#df=df.drop_duplicates()\n",
    "#drop NANs \n",
    "df=df.fillna('no data provided')\n",
    "df = df.drop_duplicates(subset='title', keep=\"first\")\n",
    "df=df[df['publish_time'].str.contains('2020')]\n",
    "# convert abstracts to lowercase\n",
    "df[\"abstract\"] = df[\"abstract\"].str.lower()+df[\"title\"].str.lower()\n",
    "#show 5 lines of the new dataframe\n",
    "df=search_focus(df)\n",
    "print (df.shape)\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import functools\n",
    "from IPython.core.display import display, HTML\n",
    "from nltk import PorterStemmer\n",
    "\n",
    "# function to stem keyword list into a common base word\n",
    "def stem_words(search_words):\n",
    "    stemmer = PorterStemmer()\n",
    "    singles=[]\n",
    "    for w in search_words:\n",
    "        singles.append(stemmer.stem(w))\n",
    "    return singles\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "def search_dataframe(df,search_words):\n",
    "    search_words=stem_words(search_words)\n",
    "    df1=df[functools.reduce(lambda a, b: a&b, (df['abstract'].str.contains(s) for s in search_words))]\n",
    "    return df1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# function analyze search results for relevance with word count / abstract length\n",
    "def search_relevance(rel_df,search_words):\n",
    "    rel_df['score']=\"\"\n",
    "    search_words=stem_words(search_words)\n",
    "    for index, row in rel_df.iterrows():\n",
    "        abstract = row['abstract']\n",
    "        result = abstract.split()\n",
    "        len_abstract=len(result)\n",
    "        score=0\n",
    "        for word in search_words:\n",
    "            score=score+result.count(word)\n",
    "        final_score=(score/len_abstract)\n",
    "        rel_score=score*final_score\n",
    "        rel_df.loc[index, 'score'] = rel_score\n",
    "    rel_df=rel_df.sort_values(by=['score'], ascending=False)\n",
    "    #rel_df= rel_df[rel_df['score'] > .01]\n",
    "    return rel_df\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# function to get best sentences from the search results\n",
    "def get_sentences(df1,search_words):\n",
    "    df_table = pd.DataFrame(columns = [\"pub_date\",\"authors\",\"title\",\"excerpt\",\"rel_score\"])\n",
    "    search_words=stem_words(search_words)\n",
    "    for index, row in df1.iterrows():\n",
    "        pub_sentence=''\n",
    "        sentences_used=0\n",
    "        #break apart the absracrt to sentence level\n",
    "        sentences = row['abstract'].split('. ')\n",
    "        #loop through the sentences of the abstract\n",
    "        highligts=[]\n",
    "        for sentence in sentences:\n",
    "            # missing lets the system know if all the words are in the sentence\n",
    "            missing=0\n",
    "            #loop through the words of sentence\n",
    "            for word in search_words:\n",
    "                #if keyword missing change missing variable\n",
    "                if word not in sentence:\n",
    "                    missing=1\n",
    "                #if '%' in sentence:\n",
    "                    #missing=missing-1\n",
    "            # after all sentences processed show the sentences not missing keywords\n",
    "            if missing==0 and len(sentence)<1000 and sentence!='':\n",
    "                sentence=sentence.capitalize()\n",
    "                if sentence[len(sentence)-1]!='.':\n",
    "                    sentence=sentence+'.'\n",
    "                pub_sentence=pub_sentence+'<br><br>'+sentence\n",
    "        if pub_sentence!='':\n",
    "            sentence=pub_sentence\n",
    "            sentences_used=sentences_used+1\n",
    "            authors=row[\"authors\"].split(\" \")\n",
    "            link=row['doi']\n",
    "            title=row[\"title\"]\n",
    "            score=row[\"score\"]\n",
    "            linka='https://doi.org/'+link\n",
    "            linkb=title\n",
    "            sentence='<p fontsize=tiny\" align=\"left\">'+sentence+'</p>'\n",
    "            final_link='<p align=\"left\"><a href=\"{}\">{}</a></p>'.format(linka,linkb)\n",
    "            to_append = [row['publish_time'],authors[0]+' et al.',final_link,sentence,score]\n",
    "            df_length = len(df_table)\n",
    "            df_table.loc[df_length] = to_append\n",
    "    return df_table\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<h1>Task 1: What is known about transmission, incubation, and environmental stability?</h1>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<h3>Results currently limited to two (10) for ease of scanning</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What is the range of incubation periods for the disease in humans?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-03-18</td>\n",
       "      <td>Jiang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.15.20036533\">Is a 14-day quarantine period optimal for effectively controlling coronavirus disease 2019 (COVID-19)?</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results the full range of incubation periods of the covid-19 cases ranged from 0 to 33 days among 2015 cases.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-15</td>\n",
       "      <td>Bui, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.09.20096800\">Estimation of the incubation period of SARS-CoV-2 in Vietnam</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Average incubation periods estimated using different distribution model ranged from 6.0 days to 6.4 days with the weibull distribution demonstrated the best fit to the data.<br><br>The estimated mean of incubation period using weibull distribution model was 6.4 days (95% credible interval (ci): 4.89 - 8.5), standard deviation (sd) was 3.05 (95%ci 3.05 - 5.30), median was 5.6, ranges from 1.35 to 13.04 days (2.5th to 97.5th percentiles).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-08</td>\n",
       "      <td>Xia, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.06.20031955\">Transmission of corona virus disease 2019 during the incubation period may lead to a quarantine loophole</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: the estimated mean incubation period for covid-19 was 4.9 days (95% confidence interval [ci], 4.4 to 5.4) days, ranging from 0.8 to 11.1 days (2.5th to 97.5th percentile).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Yang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Estimation of incubation period and serial interval of COVID-19: analysis of 178 cases and 131 transmission chains in Hubei province, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our estimated median incubation period of covid-19 is 5.4 days (bootstrapped 95% confidence interval (ci) 4.8-6.0), and the 2.5th and 97.5th percentiles are 1 and 15 days, respectively; while the estimated serial interval of covid-19 falls within the range of -4 to 13 days with 95% confidence and has a median of 4.6 days (95% ci 3.7-5.5).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Yang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[The preliminary analysis on the characteristics of the cluster for the COVID-19]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We selected 325 cases to estimate the incubation period and its range was 1 to 20 days, median was 7 days, and mode was 4 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-08</td>\n",
       "      <td>Yang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.3760/cma.j.cn112338-20200223-00153\">[The preliminary analysis on the characteristics of the cluster for the Corona Virus Disease].</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We selected 325 cases to estimate the incubation period and found its range is 1 to 20 days, median was 7 days, and mode was 4 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-01-28</td>\n",
       "      <td>Linton, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.01.26.20018754\">Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our results show that the incubation period falls within the range of 2-14 days with 95% confidence and has a mean of around 5 days when approximated using the best-fit lognormal distribution.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-01-28</td>\n",
       "      <td>Backer, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.01.27.20018986\">The incubation period of 2019-nCoV infections among travellers from Wuhan, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Using the travel history and symptom onset of 88 confirmed cases that were detected outside wuhan, we estimate the mean incubation period to be 6.4 (5.6 - 7.7, 95% ci) days, ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-02-06</td>\n",
       "      <td>Backer, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.2807/1560-7917.es.2020.25.5.2000062\">Incubation period of 2019 novel coronavirus (2019-nCoV) infections among travellers from Wuhan, China, 20–28 January 2020</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Using the travel history and symptom onset of 88 confirmed cases that were detected outside wuhan in the early outbreak phase, we estimate the mean incubation period to be 6.4 days (95% credible interval: 5.6–7.7), ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Backer, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Incubation period of 2019 novel coronavirus (2019-nCoV) infections among travellers from Wuhan, China, 20-28 January 2020</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Using the travel history and symptom onset of 88 confirmed cases that were detected outside wuhan in the early outbreak phase, we estimate the mean incubation period to be 6.4 days (95% credible interval: 5.6-7.7), ranging from 2.1 to 11.1 days (2.5th to 97.5th percentile).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-19</td>\n",
       "      <td>Kong, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/irv.12773\">Pre‐symptomatic transmission of novel coronavirus in community settings</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median incubation period was 6.0 days (interquartile range, 3.5‐9.5 days).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Kong, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Pre-symptomatic transmission of novel coronavirus in community settings</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median incubation period was 6.0 days (interquartile range, 3.5-9.5 days).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-02-23</td>\n",
       "      <td>Ai, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.02.19.20025023\">The cross-sectional study of hospitalized coronavirus disease 2019 patients in Xiangyang, Hubei province</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Incubation time ranged from one to twenty days with a mean period of 8.09 days (sd 4.99).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-24</td>\n",
       "      <td>Carcione, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.20.20072272\">A simulation of a COVID-19 epidemic based on a deterministic SEIR model</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>If we relax these constraints and use a wider range of lower and upper bounds for the incubation and infection periods, we observe that a higher incubation period (13 versus 4.25 days) gives the same ifr (0.6 versus 0.57 %), but nine times more exposed individuals in the first case.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-28</td>\n",
       "      <td>Carcione, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.3389/fpubh.2020.00230\">A Simulation of a COVID-19 Epidemic Based on a Deterministic SEIR Model</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>If we relax these constraints and use a wider range of lower and upper bounds for the incubation and infectious periods, we observe that a higher incubation period (13 vs.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: How long are individuals are contagious?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-05-23</td>\n",
       "      <td>Weiss, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.21.20108605\">Spatial and temporal dynamics of SARS-CoV-2 in COVID-19 patients: A systematic review</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In this study, we aimed to provide a coherent overview from published studies of the duration of viral detection and viral load in covid-19 patients, stratified by specimen type, clinical severity and age.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Dodds, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Model-Informed Drug Repurposing: Viral Kinetic Modeling to Prioritize Rational Drug Combinations for COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The endpoints and metrics included viral load area under the curve (auc), duration of viral shedding, and epithelial cells infected.<br><br>In addition, we observed that the time-window opportunity for a therapeutic intervention to effect duration of viral shedding exceeds the effect on sparing epithelial cells from infection or impact on viral load auc.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-21</td>\n",
       "      <td>Dodds, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/bcp.14486\">Model-Informed Drug Repurposing: Viral Kinetic Modeling to Prioritize Rational Drug Combinations for COVID-19.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The endpoints and metrics included viral load area under the curve (auc), duration of viral shedding, and epithelial cells infected.<br><br>In addition, we observed that the time-window opportunity for a therapeutic intervention to effect duration of viral shedding exceeds the effect on sparing epithelial cells from infection or impact on viral load auc.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-28</td>\n",
       "      <td>Cevik, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.07.25.20162107\">SARS-CoV-2 viral load dynamics, duration of viral shedding and infectiousness: a living systematic review and meta-analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Background viral load kinetics and the duration of viral shedding are important determinants for disease transmission.<br><br>We aim i) to characterise viral load dynamics, duration of viral rna, and viable virus shedding of sars-cov-2 in various body fluids and ii) to compare sars-cov-2 viral dynamics with sars-cov-1 and mers-cov.<br><br>Methods: medline, embase, europe pmc, preprint servers and grey literature were searched to retrieve all articles reporting viral dynamics and duration of sars-cov-2, sars-cov-1 and mers-cov shedding.<br><br>Funding: no funding was received.sars-cov-2 viral load dynamics, duration of viral shedding and infectiousness: a living systematic review and meta-analysis.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Warabi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Effects of oral care on prolonged viral shedding in coronavirus disease 2019 (COVID-19)</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods and results: we evaluated the clinical course of eight covid-19 patients, including their duration of viral shedding, by pcr testing of nasopharyngeal swabs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-24</td>\n",
       "      <td>Warabi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/scd.12498\">Effects of oral care on prolonged viral shedding in coronavirus disease 2019 (COVID-19).</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods and results we evaluated the clinical course of eight covid-19 patients, including their duration of viral shedding, by pcr testing of nasopharyngeal swabs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Chen, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Associations of Clinical Characteristics and Treatment Regimens with Viral RNA Shedding Duration in Patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: the median viral rna shedding duration was 12 days (interquartile range, 8-16 d) after the onset of illness.<br><br>Lopinavir/ritonavir use may be associated with prolonged viral rna shedding in non-severe patients; further randomized controlled trials are needed to confirm this finding.associations of clinical characteristics and treatment regimens with viral rna shedding duration in patients with covid-19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-26</td>\n",
       "      <td>Tan, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.24.20042382\">Viral Kinetics and Antibody Responses in Patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The routes and duration of viral shedding, antibody response, and their associations with disease severity and clinical manifestations were systematically evaluated.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Duration of SARS-CoV-2 RNA shedding and factors associated with prolonged viral shedding in patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A retrospective cohort of covid-19 patients admitted to a designated hospital in beijing was analyzed to study the factors affecting the duration of viral shedding.<br><br>The median duration of viral shedding was 11 days (iqr, 8-14.3 days) as measured from illness onset.<br><br>Univariate regression analysis showed that disease severity, corticosteroid therapy, fever (temperature>38.5°c), and time from onset to hospitalization were associated with prolonged duration of viral shedding (p < .05).<br><br>Multivariate regression analysis showed that fever (temperature>38.5°c) (or, 5.1, 95%ci: 1.5-18.1), corticosteroid therapy (or, 6.3, 95%ci: 1.5-27.8), and time from onset to hospitalization (or, 1.8, 95%ci: 1.19-2.7) were associated with increased odds of prolonged duration of viral shedding.<br><br>Corticosteroid treatment, fever (temperature>38.5°c), and longer time from onset to hospitalization were associated with prolonged viral shedding in covid-19 patients.duration of sars-cov-2 rna shedding and factors associated with prolonged viral shedding in patients with covid-19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-09</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1002/jmv.26280\">Duration of SARS‐CoV‐2 RNA shedding and factors associated with prolonged viral shedding in patients with COVID‐19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods: a retrospective cohort of covid‐19 patients admitted to a designated hospital in beijing was analyzed to study the factors affecting the duration of viral shedding.<br><br>Results: the median duration of viral shedding was 11 days (iqr, 8‐14.3 days) as measured from illness onset.<br><br>Univariate regression analysis showed that disease severity, corticosteroid therapy, fever (temperature>38.5℃), and time from onset to hospitalization were associated with prolonged duration of viral shedding (p<0.05).<br><br>Multivariate regression analysis showed that fever (temperature>38.5℃) (or 5.1, 95%ci: 1.5‐18.1), corticosteroid therapy (or 6.3, 95%ci: 1.5‐27.8), and time from onset to hospitalization (or 1.8, 95%ci: 1.19‐2.7) were associated with increased odds of prolonged duration of viral shedding.<br><br>All rights reserved.duration of sars‐cov‐2 rna shedding and factors associated with prolonged viral shedding in patients with covid‐19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Zhou, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Viral dynamics in asymptomatic patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>However, the duration of viral shedding remained similar in the two groups (7 days, iqr 5-14 days vs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-14</td>\n",
       "      <td>Goyal, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.10.20061325\">Potency and timing of antiviral therapy as determinants of duration of SARS CoV-2 shedding and intensity of inflammatory response</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Potent therapies dosed soon after peak viral load when infected people typically develop symptoms, are predicted to decrease shedding duration and intensity of the effector immune response, but to have little effect on viral area under the curve, which is driven by high levels of early sars cov-2 replication.<br><br>Our results support an early test and treat approach for covid-19, but also demonstrate the need to identify early viral shedding kinetic features that are the most predictive surrogates of clinical severity and transmission risk.potency and timing of antiviral therapy as determinants of duration of sars cov-2 shedding and intensity of inflammatory response.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Chan, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">A systematic review on COVID-19: urological manifestations, viral RNA detection and special considerations in urological conditions</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>5.74% (95% ci 2.88-9.44%) of covid-19 patients had positive viral rna in urine samples, but the duration of viral shedding in urine was unknown.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-30</td>\n",
       "      <td>Xu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1097/inf.0000000000002814\">Duration of Respiratory and Gastrointestinal Viral Shedding in Children With SARS-CoV-2: A Systematic Review and Synthesis of Data.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Little is known about the duration of respiratory and gastrointestinal viral shedding in children with covid-19.<br><br>Methods we performed a systematic search of ovid medline, embase and cochrane central databases for studies reporting real-time reverse transcriptase polymerase chain reaction (rt-pcr) results in children with covid-19, then extracted and synthesized data on duration of viral shedding from symptom onset in respiratory and gastrointestinal samples.<br><br>Results based on data compiled from 69 pediatric cases, the duration of viral shedding through the respiratory tract is up to 24 days from symptom onset with a mean of 11.1 ± 5.8 days.<br><br>The mean duration of viral shedding via the gastrointestinal tract was 23.6 ± 8.8 days from symptom onset.<br><br>Conclusions to our knowledge, this is the first attempt to systematically review the duration of respiratory and gastrointestinal viral shedding of sars-cov-2 in pediatric patients.<br><br>These findings may have important implications for infection control strategies during the covid-19 pandemic.duration of respiratory and gastrointestinal viral shedding in children with sars-cov-2: a systematic review and synthesis of data.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Xu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Duration of Respiratory and Gastrointestinal Viral Shedding in Children With SARS-CoV-2: A Systematic Review and Synthesis of Data</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Little is known about the duration of respiratory and gastrointestinal viral shedding in children with covid-19.<br><br>Methods: we performed a systematic search of ovid medline, embase and cochrane central databases for studies reporting real-time reverse transcriptase polymerase chain reaction (rt-pcr) results in children with covid-19, then extracted and synthesized data on duration of viral shedding from symptom onset in respiratory and gastrointestinal samples.<br><br>Results: based on data compiled from 69 pediatric cases, the duration of viral shedding through the respiratory tract is up to 24 days from symptom onset with a mean of 11.1 ± 5.8 days.<br><br>The mean duration of viral shedding via the gastrointestinal tract was 23.6 ± 8.8 days from symptom onset.<br><br>Conclusions: to our knowledge, this is the first attempt to systematically review the duration of respiratory and gastrointestinal viral shedding of sars-cov-2 in pediatric patients.<br><br>These findings may have important implications for infection control strategies during the covid-19 pandemic.duration of respiratory and gastrointestinal viral shedding in children with sars-cov-2: a systematic review and synthesis of data.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: How long are individuals are contagious, even after recovery.</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-03-31</td>\n",
       "      <td>Abduljalil, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.nmni.2020.100672\">Epidemiology, genome, and clinical features of the pandemic SARS-CoV-2: a recent view</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Asymptomatic carrier state is of paramount importance because of carriers' ability to spread the infection and to shed the virus into the air and surroundings.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Viral shedding dynamics in asymptomatic and mildly symptomatic patients infected with SARS-CoV-2</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median duration of viral shedding was 11.5, 28 and 31 days for pre-symptomatic, asymptomatic and mild symptomatic patients, separately.<br><br>Conclusions: long-term viral shedding was presented in patients with mild symptoms and asymptomatic cases, and specific antibody production to sars-cov-2 may not guarantee viral clearance after discharge.<br><br>These observations should be considered when making decisions regarding clinical and public health and strategies for prevention and control of sars-cov-2 infection.viral shedding dynamics in asymptomatic and mildly symptomatic patients infected with sars-cov-2.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Matava, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Pediatric Airway Management in COVID-19 Patients: Consensus Guidelines From the Society for Pediatric Anesthesia's Pediatric Difficult Intubation Collaborative and the Canadian Pediatric Anesthesia Society</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Pedi-c identified overarching goals during care, including minimizing aerosolized respiratory secretions, minimizing the number of clinicians in contact with a patient, and recognizing that undiagnosed asymptomatic patients may shed the virus and infect health care workers.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-20</td>\n",
       "      <td>Matava, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1213/ane.0000000000004872\">Pediatric Airway Management in Coronavirus Disease 2019 Patients: Consensus Guidelines From the Society for Pediatric Anesthesia’s Pediatric Difficult Intubation Collaborative and the Canadian Pediatric Anesthesia Society</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Pedi-c identified overarching goals during care, including minimizing aerosolized respiratory secretions, minimizing the number of clinicians in contact with a patient, and recognizing that undiagnosed asymptomatic patients may shed the virus and infect health care workers.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-25</td>\n",
       "      <td>Beggs, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.22.20109991\">Is there an airborne component to the transmission of COVID-19? : a quantitative analysis study</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Conclusions given that live sars-cov-2 virions are known to be shed in high concentrations from the nasal cavity of both symptomatic and asymptomatic covid-19 patients, the results suggest that individuals who share enclosed spaces with an infector may be at risk of contracting covid-19 by the aerosol route, even when practicing social distancing.is there an airborne component to the transmission of covid-19? : a quantitative analysis study.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Miyamae, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Duration of viral shedding in asymptomatic or mild cases of novel coronavirus disease 2019 (COVID-19) from a cruise ship: A single-hospital experience in Tokyo, Japan</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We examined the course of pcr assays and the duration of viral shedding in 23 asymptomatic or mild covid-19 patients from the cruise ship who were admitted to our hospital.<br><br>Although the duration of viral shedding was approximately three weeks, the infectivity and transmissibility period from asymptomatic and mild covid-19 cases is unclear.<br><br>Further studies are needed to determine how long such asymptomatic and mild covid-19 cases have infectivity.duration of viral shedding in asymptomatic or mild cases of novel coronavirus disease 2019 (covid-19) from a cruise ship: a single-hospital experience in tokyo, japan.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Fung, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">A tug-of-war between severe acute respiratory syndrome coronavirus 2 and host antiviral defence: lessons from other pathogenic viruses</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Important questions concerning the interaction between sars-cov-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.a tug-of-war between severe acute respiratory syndrome coronavirus 2 and host antiviral defence: lessons from other pathogenic viruses.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Khan, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Repurposing 0.5% povidone iodine solution in otorhinolaryngology practice in Covid 19 pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A significant proportion of covid 19 sufferers are asymptomatic, but shedding these viral particles, pvp-i has been shown to be a safe therapy when used as a mouthwash or taken nasally or used during ophthalmic surgeries.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Nahshon, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Hazardous Postoperative Outcomes of Unexpected COVID-19 Infected Patients: A Call for Global Consideration of Sampling all Asymptomatic Patients Before Surgical Treatment</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Few reports were published on the outcome of surgical procedures in diagnosed covid-19 patients and even fewer on the surgical outcomes of asymptomatic undiagnosed covid-19 surgical patients.<br><br>We aimed to review all published data regarding surgical outcomes of preoperatively asymptomatic untested coronavirus disease 2019 (covid-19) patients.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-01</td>\n",
       "      <td>Kern, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1055/a-1114-3699\">[SARS-CoV-2 infection (COVID-19): what can we expect?]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>- case numbers in china are clearly declining, case numbers in many european regions are no longer increasing exponentially.- data on mortality from sars-cov-2 infection are contradictory; mortality is certainly lower than for sars and mers, but probably higher than for most seasonal flu outbreaks in recent years- the main complication of sars-cov-2 infection is pneumonia with development of acute respiratory distress syndrome (ards)- asymptomatic and oligosymptomatic courses with virus shedding are not uncommon; they may be more frequent in children than in adults.<br><br>Virus excretion in asymptomatic people and in the pre-symptomatic phase of an infection is relevant for transmission- an effective antiviral therapy has not yet been established.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Park, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Detection of SARS-CoV-2 in Fecal Samples from Patients with Asymptomatic and Mild COVID-19 in Korea</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We investigated gastrointestinal symptoms and shedding of virus into feces of patients with asymptomatic or mild covid-19.<br><br>Methods: we collected data from 46 patients (median age, 26 years; 46% men) with asymptomatic or mild covid-19 (without fever and pneumonia) and prolonged respiratory shedding of sars-cov-2, quarantined from april 4, 2020 through april 24, 2020 in korea.<br><br>The viral load of the respiratory specimens appears be related to shedding of the virus into feces in this group of patients.detection of sars-cov-2 in fecal samples from patients with asymptomatic and mild covid-19 in korea.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Cheng, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Diagnostic Testing for Severe Acute Respiratory Syndrome-Related Coronavirus 2: A Narrative Review</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Although excellent tools exist for the diagnosis of symptomatic patients in well-equipped laboratories, important gaps remain in screening asymptomatic persons in the incubation phase, as well as in the accurate determination of live viral shedding during convalescence to inform decisions to end isolation.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Kirshblum, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Screening testing for SARS-CoV-2 upon admission to rehabilitation hospitals in a high COVID-19 prevalence community</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>To maintain a safe environment for all patients and staff, we established an admission screening plan of testing for sars-cov-2 to determine the presence of asymptomatic patients who were infected with the virus upon admission.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Huff, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Asymptomatic transmission during the COVID-19 pandemic and implications for public health strategies</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Viral load of asymptomatic carriers is comparable to symptomatic patients, viral shedding is highest before symptom onset suggesting high transmissibility before symptoms.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Lavezzo, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Suppression of a SARS-CoV-2 outbreak in the Italian municipality of Vo'</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This study sheds new light on the frequency of asymptomatic sars-cov-2 infection, their infectivity (as measured by the viral load) and provides new insights into its transmission dynamics and the efficacy of the implemented control measures.suppression of a sars-cov-2 outbreak in the italian municipality of vo'.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: Does the range of incubation period vary across age groups?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-02-29</td>\n",
       "      <td>Han, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.02.24.20027474\">Estimate the incubation period of coronavirus 2019 (COVID-19)</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We found that the incubation periods of the groups with age>=40 years and age<40 years demonstrated a statistically significant difference.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: Does the range of incubation period vary with children?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-03-18</td>\n",
       "      <td>Jiang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.15.20036533\">Is a 14-day quarantine period optimal for effectively controlling coronavirus disease 2019 (COVID-19)?</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median incubation period of both male and female adults was similar (7-day) but significantly shorter than that (9-day) of child cases (p=0.02).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Merino-Navarro, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Prevención y tratamiento del Covid-19 en la población pediátrica desde una perspectiva familiar y comunitaria./ [Prevention and treatment of Covid-19 in the pediatric population from the family and community perspective]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our objective is to analyze the scientific evidence on the specific recommendations for pediatric care in cases of covid-19 from the family and community settings.the main recommendations and preventive measures in primary health care settings and at home have been selected and analyzed from an integrative approach that includes the biopsychosocial aspects of the child during confinement.the importance of caring for children in the face of the disease lies above all in ensuring the correct measures for the prevention of contagion due to the condition of acting as possible carriers during an incubation period of up to 21 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-16</td>\n",
       "      <td>Merino-Navarro, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.enfcli.2020.05.005\">Prevención y tratamiento del Covid-19 en la población pediátrica desde una perspectiva familiar y comunitaria</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The importance of caring for children in the face of the disease lies above all in ensuring the correct measures for the prevention of contagion due to the condition of acting as possible carriers during an incubation period of up to 21 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Liu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Risk factors associated with COVID-19 infection: a retrospective cohort study based on contacts tracing</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Children, old people, females, and family members are susceptible of covid-19 infection, while index cases in the incubation period had lower contagiousness.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-01</td>\n",
       "      <td>Liu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1080/22221751.2020.1787799\">Risk factors associated with COVID-19 infection: a retrospective cohort study based on contacts tracing.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Conclusion children, old people, females and family members are susceptible to be infected with covid-19, while index cases in incubation period had lower contagiousness.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-01</td>\n",
       "      <td>Paglia, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.23804/ejpd.2020.21.02.01\">COVID-19 and Paediatric Dentistry after the lockdown.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The average incubation period is about 5 days, with an estimated range from 2 to 14 days; the incubation period in children is similar, however some have exhibited a longer incubation.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Paglia, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">COVID-19 and Paediatric Dentistry after the lockdown</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The average incubation period is about 5 days, with an estimated range from 2 to 14 days; the incubation period in children is similar, however some have exhibited a longer incubation.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Shen, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Novel coronavirus infection in children outside of Wuhan, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Six children had a family exposure and could provide the exact dates of close contact with someone who was confirmed to have 2019-ncov infection, among whom the median incubation period was 7.5 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Han, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">A comparative-descriptive analysis of clinical characteristics in 2019-coronavirus-infected children and adults</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median incubation period of children and adults was 5 days (ranged, 3-12 days) and 4 days (ranged, 2-12 days), respectively.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-06</td>\n",
       "      <td>Han, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1002/jmv.25835\">A comparative-descriptive analysis of clinical characteristics in 2019-Coronavirus-infected children and adults.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median incubation period of children and adults was 5 days (range 3-12 days) and 4 days (range 2-12 days), respectively.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Di et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">A literature review of 2019 novel coronavirus (SARS-CoV2) infection in neonates and children</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Impact: children usually develop a mild form of covid-19, rarely requiring high-intensity medical treatment in pediatric intensive care unit.vertical transmission is unlikely, but not completely excluded.children with confirmed or suspected covid-19 must be isolated and healthcare workers should wear appropriate protective equipment.some clinical features (higher incidence of fever, vomiting and diarrhea, and a longer incubation period) are more common in children than in adults, as well as some radiologic aspects (more patchy shadow opacities on ct scan images than ground-glass opacities).supportive and symptomatic treatments (oxygen therapy and antibiotics for preventing/treating bacterial coinfections) are recommended in these patients.a literature review of 2019 novel coronavirus (sars-cov2) infection in neonates and children.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Zhang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.21037/tp.2020.03.08\">Analysis and suggestions for the preview and triage screening of children with suspected COVID-19 outside the epidemic area of Hubei Province</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The main reason for this spread is the highly contagious nature of the virus and the fact that children can also become infected during its incubation period.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-17</td>\n",
       "      <td>Di et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1038/s41390-020-1065-5\">A literature review of 2019 novel coronavirus (SARS-CoV2) infection in neonates and children.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Some clinical features (higher incidence of fever, vomiting and diarrhea, and a longer incubation period) are more common in children than in adults, as well as some radiologic aspects (more patchy shadow opacities on ct scan images than ground-glass opacities).</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: Does the range of incubation period vary based on underlying health?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-04-13</td>\n",
       "      <td>Saakian, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">A simple statistical physics model for the epidemic with incubation period</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We propose a strategy: organize a quarantine, and then conduct extensive testing of risk groups during the quarantine, evaluating the percentage of the population among risk groups and people with symptoms.a simple statistical physics model for the epidemic with incubation period.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about the basic reproduction number?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-02-20</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.02.18.20024315\">Estimation of the epidemic properties of the 2019 novel coronavirus: A mathematical modeling study</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Findings before the closure of wuhan city the basic reproduction number of covid-19 was 4.38 (95% ci: 3.63-5.13), dropping to 3.41 (95% ci: 3.16-3.65) after the closure of wuhan city.<br><br>Over the entire epidemic period covid-19 had a basic reproduction number of 3.39 (95% ci: 3.09-3.70), indicating it has a very high transmissibility.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-30</td>\n",
       "      <td>Marimuthu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.cegh.2020.06.012\">Modelling of reproduction number for COVID-19 in India and high incidence states</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: the basic reproduction number (r(0)) for india was estimated at 1.379 (95% ci: 1.375, 1.384).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-20</td>\n",
       "      <td>Russo, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.17.20037689\">Tracing DAY-ZERO and Forecasting the Fade out of the COVID-19 Outbreak in Lombardy, Italy: A Compartmental Modelling and Numerical Optimization Approach.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The \"effective\" per-day disease transmission rate for the period until march 8 was found to be 0.686 (95% ci:0.660, 0.713), while the basic reproduction number r0 was found to be 4.51 (95% ci: 4.14, 4.90).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-20</td>\n",
       "      <td>Jia, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.18.20038570\">Extended SIR prediction of the epidemics trend of COVID-19 in Italy and compared with Hunan, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The basic reproductive number was estimated using markov chain monte carlo methods and presented using the resulting posterior mean and 95% credible interval (ci).<br><br>Results: in the esir model, we estimated that the basic reproductive number for covid 19 was respectively 4.10 (95% ci: 2.15 to 6.77) in italy and 3.15(95% ci: 1.71 to 5.21) in hunan.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Wangping, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.3389/fmed.2020.00169\">Extended SIR Prediction of the Epidemics Trend of COVID-19 in Italy and Compared With Hunan, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The basic reproductive number was estimated using markov chain monte carlo methods and presented using the resulting posterior mean and 95% credible interval (ci).<br><br>Results: in the esir model, we estimated that the mean of basic reproductive number for covid-19 was 4.34 (95% ci, 3.04–6.00) in italy and 3.16 (95% ci, 1.73–5.25) in hunan.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Song, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[Study on assessing early epidemiological parameters of COVID-19 epidemic in China]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The basic reproduction number was estimated to be 3.74 (95%ci: 3.63-3.87), 3.16 (95%ci: 2.90-3.43), and 3.91 (95%ci: 3.71-4.11) by three methods, respectively.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-01</td>\n",
       "      <td>Song, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.3760/cma.j.cn112338-20200205-00069\">[Study on assessing early epidemiological parameters of coronavirus disease epidemic in China].</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The basic reproduction number was estimated to be 3.74 (95%ci: 3.63-3.87), 3.16 (95%ci: 2.90-3.43), and 3.91 (95%ci: 3.71-4.11) by three methods, respectively.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-11</td>\n",
       "      <td>Liu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.07.20094086\">Transmission in Latent Period Causes A Large Number of Infected People in the United States</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The results show that the basic reproduction number in the early period of propagation in the united states is estimated to be 4.06 (95% ci: 1.86-6.73) based on the confirmed cases data ranging from january 21, 2020 to march 21, 2020.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Kim, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Risk estimation of the SARS-CoV-2 acute respiratory disease outbreak outside China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The rate of transmission was estimated as 0.8238 (95% confidence interval [ci] 0.8095-0.8382), and the basic reproductive number as 4.1192 (95% ci 4.0473-4.1912).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-20</td>\n",
       "      <td>Cereda, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">The early phase of the COVID-19 outbreak in Lombardy, Italy</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We estimate the basic reproduction number at 3.1 (95% ci, 2.9 to 3.2).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Musa, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Estimation of exponential growth rate and basic reproduction number of the coronavirus disease 2019 (COVID-19) in Africa</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: we estimated the exponential growth rate as 0.22 per day (95% ci: 0.20-0.24), and the basic reproduction number, r0, as 2.37 (95% ci: 2.22-2.51) based on the assumption that the exponential growth starting from 1 march 2020.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-27</td>\n",
       "      <td>Wenbao, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.24.20036285\">Transmission dynamics of SARS-COV-2 in China: impact of public health interventions</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our results showed that the seird curve model we established had a good fit and the basic reproductive number is 3.38 (95% ci, 3.25-3.48).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-29</td>\n",
       "      <td>Mukandavire, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.23.20077297\">Quantifying early COVID-19 outbreak transmission in South Africa and exploring vaccine efficacy scenarios</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: early model estimates show that covid-19 outbreak in south africa had a basic reproductive number of 2.95 (95% credible interval [cri] 2.83-3.33).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>He, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for COVID-19: Meta-analysis and sensitivity analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This manuscript conducts a meta-analysis and a sensitivity study to answer the questions: what is the basic reproduction number? how long is the incubation time of the disease on average? what portion of infections are asymptomatic? and ultimately, what is the case fatality rate? our studies estimate the basic reproduction number to be 3.15 with the 95% ci (2.41-3.90), the average incubation time to be 5.08 days with the 95% ci (4.77-5.39) (in day), the asymptomatic infection rate to be 46% with the 95% ci (18.48%-73.60%), and the case fatality rate to be 2.72% with 95% ci (1.29%-4.16%) where asymptomatic infections are accounted for.estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for covid-19: meta-analysis and sensitivity analysis.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-09</td>\n",
       "      <td>He, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1002/jmv.26041\">Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for COVID‐19: Meta‐analysis and sensitivity analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This manuscript conducts a meta‐analysis and a sensitivity study to answer the questions: what is the basic reproduction number? how long is the incubation time of the disease on average? what portion of infections are asymptomatic? and ultimately, what is the case fatality rate? our studies estimate the basic reproduction number to be 3.15 with the 95% ci (2.41‐3.90), the average incubation time to be 5.08 days with the 95% ci (4.77‐5.39) (in day), the asymptomatic infection rate to be 46% with the 95% ci (18.48%‐73.60%), and the case fatality rate to be 2.72% with 95% ci (1.29%‐4.16%) where asymptomatic infections are accounted for.estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for covid‐19: meta‐analysis and sensitivity analysis.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What is the prevalance of asymptomatic transmission?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-06-19</td>\n",
       "      <td>Ling, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.16.20132423\">Seroprevalence and epidemiological characteristics of immunoglobulin M and G antibodies against SARS-CoV-2 in asymptomatic people in Wuhan, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Based on the census population aged 16-64 years in wuhan in 2017, we estimated that 172,340 (95% ci: 157,568-187,112) asymptomatic people aged 16-64 years were infected with sars-cov-2 in wuhan between march 25 and april 28, 2020.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Hao, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Reconstruction of the full transmission dynamics of COVID-19 in Wuhan</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We estimate 87% (lower bound, 53%) of the infections before 8 march 2020 were unascertained (potentially including asymptomatic and mildly symptomatic individuals); and a basic reproduction number (r0) of 3.54 (95% credible interval 3.40-3.67) in the early outbreak, much higher than that of severe acute respiratory syndrome (sars) and middle east respiratory syndrome (mers)4,5.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Park, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Coronavirus Disease Outbreak in Call Center, South Korea</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Of the 97 persons with confirmed covid-19, only 4 (1.9%) remained asymptomatic within 14 days of quarantine, and none of their household contacts acquired secondary infections.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-16</td>\n",
       "      <td>Friston, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Tracking and tracing in the UK: a dynamic causal modelling study</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Crucially, this deferment is within current testing capabilities (requiring an efficacy of tracing and tracking of about 20% of asymptomatic infected cases, with less than 50,000 tests per day).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Sandmann, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Optimising benefits of testing key workers for infection with SARS-CoV-2: A mathematical modelling analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We varied all parameters, including the infection rate (0.1%-20%), proportion asymptomatic (10%-80%), sensitivity (60%-95%), and specificity (90%-100%).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Rokni, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Immune responses and pathogenesis of SARS-CoV-2 during an outbreak in Iran: Comparison with SARS and MERS</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In worldwide, the incubation period of covid-19 was 3 to 7 days and approximately 80% of infections are mild or asymptomatic, 15% are severe, requiring oxygen, and 5% are critical infections, requiring ventilation.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Lavezzo, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Suppression of a SARS-CoV-2 outbreak in the Italian municipality of Vo'</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Notably, 42.5% (95% ci: 31.5-54.6%) of the confirmed sars-cov-2 infections detected across the two surveys were asymptomatic (that is, did not have symptoms at the time of swab testing and did not develop symptoms afterwards).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Zhang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[Investigation on a cluster epidemic of COVID-19 in a supermarket in Liaocheng, Shandong province]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A total 25 cases of sars-cov-2 infection, the total infection rate of subjects was 0.30% (25/8 437) with 22 confirmed cases (0.26%, 22/8 437) and 3 asymptomatic patients (0.04%, 3/8 437), asymptomatic patients accounted for 12.00% (3/25) of all infection cases.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Wang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Clinical characteristics and laboratory results of pregnant women with COVID-19 in Wuhan, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Compared with the nonpregnant group, pregnant patients were admitted to hospital earlier (0.25 vs 11.00 days; p<0.001), presented milder symptoms, had a higher rate of asymptomatic infection (26.7% vs 0%), and shorter length of hospital stay (14.5 vs 17.0 days; p<0.01).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Louie, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Lessons from Mass-Testing for COVID-19 in Long Term Care Facilities for the Elderly in San Francisco</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We describe four ltcf outbreaks where mass testing identified a high proportion of asymptomatic infections (4-41% in health care workers and 20-75% in residents), indicating that symptom-based screening alone is insufficient for monitoring for covid-19 transmission.lessons from mass-testing for covid-19 in long term care facilities for the elderly in san francisco.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Cavalli, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Entangling COVID-19 associated thrombosis into a secondary antiphospholipid antibody syndrome: Diagnostic and therapeutic perspectives (Review)</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Most (some 80%) cases of covid­19 infection are asymptomatic, a substantial number of cases (15%) require hospitalization and an additional fraction of patients (5%) need recovery in intensive care units.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Oran, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Prevalence of Asymptomatic SARS-CoV-2 Infection: A Narrative Review</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Asymptomatic persons seem to account for approximately 40% to 45% of sars-cov-2 infections, and they can transmit the virus to others for an extended period, perhaps longer than 14 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Jing, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[Contagiousness and secondary attack rate of 2019 novel coronavirus based on cluster epidemics of COVID-19 in Guangzhou]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: as of 18 february, 2020, a total of 349 cases of 2019-ncov infection, including 339 confirmed cases (97.13%) and 10 asymptomatic cases (2.87%) were reported in guangzhou.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Bianco, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Testing of Patients and Support Persons for Coronavirus Disease 2019 (COVID-19) Infection Before Scheduled Deliveries</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The prevalence of asymptomatic covid-19 infection was 15.5% (ci 9.8-21.2%) and 9.6% (ci 4.8-14.4%) among patients and support persons, respectively.<br><br>Conclusion: we found that more than 15% of asymptomatic maternity patients tested positive for sars-cov-2 infection despite having screened negative with the use of a telephone screening tool.<br><br>Additionally, 58% of their asymptomatic, screen-negative support persons also tested positive for sars-cov-2 infection.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>He, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for COVID-19: Meta-analysis and sensitivity analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This manuscript conducts a meta-analysis and a sensitivity study to answer the questions: what is the basic reproduction number? how long is the incubation time of the disease on average? what portion of infections are asymptomatic? and ultimately, what is the case fatality rate? our studies estimate the basic reproduction number to be 3.15 with the 95% ci (2.41-3.90), the average incubation time to be 5.08 days with the 95% ci (4.77-5.39) (in day), the asymptomatic infection rate to be 46% with the 95% ci (18.48%-73.60%), and the case fatality rate to be 2.72% with 95% ci (1.29%-4.16%) where asymptomatic infections are accounted for.estimation of the basic reproduction number, average incubation time, asymptomatic infection rate, and case fatality rate for covid-19: meta-analysis and sensitivity analysis.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about asymptomatic transmission in children?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-05-01</td>\n",
       "      <td>Chen, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[Asymptomatic SARS-CoV-2 infection in children: a clinical analysis of 20 cases].</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Objective to study the clinical and epidemiological features of children with asymptomatic severe acute respiratory syndrome coronavirus 2 (sars-cov-2) infection.<br><br>Methods the clinical data of 20 children who were diagnosed with asymptomatic sars-cov-2 infection from january 20 to march 4, 2020 were analyzed.<br><br>Most of the children with asymptomatic infection have no obvious abnormalities in blood routine and other laboratory tests.<br><br>Changes in chest ct scan can be used as an aid for early diagnosis of asymptomatic infection in children.[asymptomatic sars-cov-2 infection in children: a clinical analysis of 20 cases].</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Chen, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[Asymptomatic SARS-CoV-2 infection in children: a clinical analysis of 20 cases]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Objective: to study the clinical and epidemiological features of children with asymptomatic severe acute respiratory syndrome coronavirus 2 (sars-cov-2) infection.<br><br>Methods: the clinical data of 20 children who were diagnosed with asymptomatic sars-cov-2 infection from january 20 to march 4, 2020 were analyzed.<br><br>Most of the children with asymptomatic infection have no obvious abnormalities in blood routine and other laboratory tests.<br><br>Changes in chest ct scan can be used as an aid for early diagnosis of asymptomatic infection in children.[asymptomatic sars-cov-2 infection in children: a clinical analysis of 20 cases].</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-24</td>\n",
       "      <td>Xu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.20.20073288\">A follow-up study of children infected with SARS-CoV-2 from Western China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Background: to clarify the characteristic and the duration of positive nucleic acid in children infected with severe acute respiratory syndrome coronavirus 2 (sars-cov-2), including asymptomatic children.<br><br>Results: eleven children (34%) were asymptomatic, among whom six children had normal computed tomographic (ct) scan images.<br><br>The concentrations of white blood cells and neutrophils were higher in children with asymptomatic infection than in children with clinical symptoms or ct abnormalities.<br><br>The mean duration of positive sars-cov-2 nucleic acid was 15.4 (sd=7.2) days and similar for both asymptomatic children and children with symptoms or ct abnormalities.<br><br>Conclusions: children with asymptomatic infection should be quarantined for the same duration as symptomatic patients infected with sars-cov-2.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-01</td>\n",
       "      <td>Xu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.21037/atm-20-3192\">A follow-up study of children infected with SARS-CoV-2 from western China.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Background to clarify the characteristic and the duration of positive nucleic acid in children infected with severe acute respiratory syndrome coronavirus 2 (sars-cov-2), including asymptomatic children.<br><br>Results eleven children (34%) were asymptomatic, among whom six children had normal computed tomographic (ct) scan images.<br><br>The concentrations of white blood cells and neutrophils were higher in children with asymptomatic infection than in children with clinical symptoms or ct abnormalities.<br><br>The mean duration of positive sars-cov-2 nucleic acid was 15.4 (sd =7.2) days and similar for both asymptomatic children and children with symptoms or ct abnormalities.<br><br>Conclusions children with asymptomatic infection should be quarantined for the same duration as symptomatic patients infected with sars-cov-2.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-01</td>\n",
       "      <td>Ma, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[Clinical features of children with SARS-CoV-2 infection: an analysis of 115 cases].</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>As for clinical symptoms, 29 children (25.2%) had fever, 47 (40.9%) had respiratory symptoms (including cough, rhinorrhea, and nasal congestion), and 61 (53.0%) were asymptomatic.<br><br>Critical cases are rare and there is a high proportion of asymptomatic infection.[clinical features of children with sars-cov-2 infection: an analysis of 115 cases].</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Ma, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[Clinical features of children with SARS-CoV-2 infection: an analysis of 115 cases]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>As for clinical symptoms, 29 children (25.2%) had fever, 47 (40.9%) had respiratory symptoms (including cough, rhinorrhea, and nasal congestion), and 61 (53.0%) were asymptomatic.<br><br>Critical cases are rare and there is a high proportion of asymptomatic infection.[clinical features of children with sars-cov-2 infection: an analysis of 115 cases].</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-24</td>\n",
       "      <td>Lu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/pai.13267\">SARS-CoV-2 infection in children - Understanding the immune responses and controlling the pandemic.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The burden of disease in children has been relatively low, but the high proportions of asymptomatic or mildly symptomatic infections in children deserve careful attention.<br><br>Clear understanding of the immune responses to the virus in children and the transmission potential of asymptomatic children are of paramount importance for the development of specific treatments and vaccine in order to effectively control the ongoing pandemic.sars-cov-2 infection in children - understanding the immune responses and controlling the pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-01</td>\n",
       "      <td>Wang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.21037/atm-20-3302\">Clinical characteristics of children with COVID-19: a rapid review and meta-analysis.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Conclusions most children with covid-19 have only mild symptoms, and many children are asymptomatic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-17</td>\n",
       "      <td>Wang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.13.20064352\">Clinical Characteristics of Children with COVID-19: A Rapid Review and Meta-Analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Conclusions: most children with covid-19 have only mild symptoms, and many children are asymptomatic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Du, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Clinical characteristics of COVID-19 in children compared with adults in Shandong Province, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>There were 8 (57.1%) asymptomatic cases and 6 (42.9%) symptomatic cases among the 14 children cases.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-25</td>\n",
       "      <td>Su, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1080/22221751.2020.1744483\">The different clinical characteristics of corona virus disease cases between children and their families in China – the character of children with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Because of the mild or asymptomatic clinical process, it is difficult to recognize early for pediatrician and public health staff.the different clinical characteristics of corona virus disease cases between children and their families in china – the character of children with covid-19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Su, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">The different clinical characteristics of corona virus disease cases between children and their families in China - the character of children with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Because of the mild or asymptomatic clinical process, it is difficult to recognize early for pediatrician and public health staff.the different clinical characteristics of corona virus disease cases between children and their families in china - the character of children with covid-19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Lu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">COVID-19: lessons to date from China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Older children are more likely to have asymptomatic infection.<br><br>Because a large proportion of children and adolescents may have asymptomatic or mildly symptomatic infection, children are likely to play an important role in community transmission of this infection.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-12</td>\n",
       "      <td>Lu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1136/archdischild-2020-319261\">COVID-19: lessons to date from China.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Older children are more likely to have asymptomatic infection.<br><br>Because a large proportion of children and adolescents may have asymptomatic or mildly symptomatic infection, children are likely to play an important role in community transmission of this infection.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Lu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">SARS-CoV-2 infection in children - Understanding the immune responses and controlling the pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The burden of disease in children has been relatively low, but the high proportions of asymptomatic or mildly symptomatic infections in children deserve careful attention.<br><br>A clear understanding of the immune responses to the virus in children and the transmission potential of asymptomatic children is of paramount importance for the development of specific treatments and vaccine in order to effectively control the ongoing pandemic.sars-cov-2 infection in children - understanding the immune responses and controlling the pandemic.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about seasonality of transmission?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Itaya, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Does COVID-19 infection impact on the trend of seasonal influenza infection? 11 countries and regions, from 2014 to 2020</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: in east asia, the number of seasonal influenza cases in the 2019-20 season was lower after the covid-19 transmission compared to previous years.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-02</td>\n",
       "      <td>Douglas, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.3390/tropicalmed5020068\">Dengue Fever and Severe Dengue in Barbados, 2008–2016</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Df cases occurred throughout the year with low level of transmission observed during the dry season (december to june), then increased transmission during rainy season (july to november) peaking in october.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-01</td>\n",
       "      <td>Magurano, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.30.20099143\">SARS-CoV-2 infection: the environmental endurance of the virus can be influenced by the increase of temperature</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our results support the hypothesis that in the hot season the increase of temperature could influence the environmental endurance of sars-cov2 and reduce covid-19 transmission probability.sars-cov-2 infection: the environmental endurance of the virus can be influenced by the increase of temperature.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-19</td>\n",
       "      <td>Huang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.07.17.20156430\">Transmission Dynamics of Coronavirus Disease 2019 (COVID-19) in the World: The Roles of Intervention and Seasonality</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In this study, we developed a novel compartmental model, seir-cv(susceptible-exposed-infectious-removed with control variables), which not only considers the key characteristics of asymptomatic infection and the effects of seasonal variations, but also incorporates different control measures for multiple transmission routes, so as to accurately predict and effectively control the spread of covid-19.<br><br>However, if china's super stringent control measures were implemented from 15 july, 15 august or 15 september 2020, the total infections would be contained about 15 million, 32 million or 370 million respectively, which indicates that the stringent and timely control measures is critical, and the best window period is before september for eventually overcoming covid-19.transmission dynamics of coronavirus disease 2019 (covid-19) in the world: the roles of intervention and seasonality.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-28</td>\n",
       "      <td>Müller, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.27.052225\">Characterising the epidemic spread of Influenza A/H3N2 within a city through phylogenetics</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The genetic sequence data allows us to reconstruct the how the transmission dynamics changed over the course of the season, which we correlate to changes, but not humidity or school holidays.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-09</td>\n",
       "      <td>Neher, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.4414/smw.2020.20224\">Potential impact of seasonal forcing on a SARS-CoV-2 pandemic.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Here, we explore how seasonal variation in transmissibility could modulate a sars-cov-2 pandemic.<br><br>Seasonal forcing on sars-cov-2 should thus be taken into account in the further monitoring of the global transmission.<br><br>The likely aggregated effect of seasonal variation, infection control measures, and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.potential impact of seasonal forcing on a sars-cov-2 pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-02-17</td>\n",
       "      <td>Neher, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.02.13.20022806\">Potential impact of seasonal forcing on a SARS-CoV-2 pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Here, we explore how seasonal variation in transmissibility could modulate a sars-cov-2 pandemic.<br><br>Seasonal forcing on sars-cov-2 should thus be taken into account in the further monitoring of the global transmission.<br><br>The likely aggregated effect of seasonal variation, infection control measures and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.potential impact of seasonal forcing on a sars-cov-2 pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Pramanik, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Climatic factors influence the spread of COVID-19 in Russia</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The results show that temperature seasonality (29.2 ± 0.9%) has the highest contribution for covid-19 transmission in the humid continental region.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-28</td>\n",
       "      <td>Muñoz, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1038/s41598-020-69625-4\">AeDES: a next-generation monitoring and forecasting system for environmental suitability of Aedes-borne disease transmission</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>After analyzing the seasonal predictive skill of aedes, we briefly consider the recent zika epidemic, and the compound effects of the current central american dengue outbreak happening during the sars-cov-2 pandemic, to illustrate how a combination of tailored deterministic and probabilistic forecasts can inform key prevention and control strategies .aedes: a next-generation monitoring and forecasting system for environmental suitability of aedes-borne disease transmission.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Dong, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Infectious diseases in children and adolescents in China: analysis of national surveillance data from 2008 to 2017</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Many challenges remain around reducing regional inequalities, scaling-up of vaccination, prevention of further escalation of hiv/aids, renewed efforts for persisting diseases, and undertaking early and effective response to highly transmissible seasonal and unpredictable diseases such as that caused by the novel sars-cov-2 virus.infectious diseases in children and adolescents in china: analysis of national surveillance data from 2008 to 2017.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-14</td>\n",
       "      <td>Hoffman, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.08.20095505\">Significant Relaxation of SARS-CoV-2-Targeted Non-Pharmaceutical Interventions Will Result in Profound Mortality: A New York State Modelling Study</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The aim of this study was to develop a model to estimate the effect of undocumented infections, seasonal infectivity, immunity, and non-pharmaceutical interventions (npis), such as social distancing, on the transmission, morbidity, and mortality of sars-cov-2 in new york state (nys).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-26</td>\n",
       "      <td>Raines, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.23.20111278\">The transmission of SARS-CoV-2 is likely comodulated by temperature and by relative humidity</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In this paper we focus our analysis on data from colombia, which presents unique economic, demographic and geological characteristics that favor the study of temperature and humidity upon sars-cov-2 transmission: the weather varies dramatically across five natural regions (from the caribbean coast and the amazon rainforest to the andean mountains), there are no pronounced seasons, there is a central port of entry, the use of public transportation dominates inter- and intracity travel, and indoor climate control is rare.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-16</td>\n",
       "      <td>Zhang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.16.154591\">Antigenic evolution on global scale reveals potential natural selection of SARS-CoV-2 by pre-existing cross-reactive T cell immunity</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Further analysis revealed mutations in cross-reactive epitopes between sars-cov-2 and seasonal coronavirus may help sars-cov-2 to escape cellular immunity under the long-term and large-scale community transmission.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-16</td>\n",
       "      <td>Telles, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.07.15.20154823\">SARS-COV-2 THREE FORCING SEASONALITIES: POLICIES, ENVIRONMENT AND URBAN SPACES</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Putting in order {varepsilon}' of influence, there were identified three main forms of sars-cov-2 of transmission behavior: during epidemics growth, policies are the main stronger seasonality forcing behavior of the epidemics followed by secondary and weaker environmental and urban spaces driving patterns of transmission.<br><br>And also, it causes policies/ale limitations to address urban spaces and environmental seasonality instabilities, thus generating posterior waves or uncontrolled patterns of transmission (fluctuations).<br><br>All this components affect the sars-cov-2 spreading patterns simultaneously being not possible to observe environmental seasonality not associated intrinsically with policies/ale and urban spaces, therefore conferring to these three forms of transmission spreading patterns, specific regions of analysis for time series data extraction.sars-cov-2 three forcing seasonalities: policies, environment and urban spaces.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-28</td>\n",
       "      <td>Ribeiro, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1002/jmv.26079\">Arboviral diseases and COVID‐19 in Brazil: Concerns regarding climatic, sanitation and endemic scenario</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Brazil is witnessing a massive increase of covid‐19 cases and may face some difficulties, not only regarding to other respiratory pathogens, but also to other relevant issues parallelly occurring, for instance, the beggining of autumn and winter seasons, which provides a longer period with high transmissibility of respiratory viroses.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: Informing decontamination based on physical science of the coronavirus?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Lv, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Detection of SARS-CoV-2 RNA residue on object surfaces in nucleic acid testing laboratory using droplet digital PCR</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We found that all the positive objects were directly or indirectly contacted by the operator's gloved hands, suggesting that hands contact was the main transmission pathway that led to laboratory environmental contamination.<br><br>We evaluated the risk areas and operation behaviors that may easily cause contamination, and provided recommendation for improving the laboratory disinfection programs and personal operating specifications.detection of sars-cov-2 rna residue on object surfaces in nucleic acid testing laboratory using droplet digital pcr.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Couloigner, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">COVID-19 and ENT Surgery</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In otorhinolaryngology - head and neck surgery, clinical examination and invasive procedures on the respiratory tract and on airway-connected cavities, such as paranasal sinuses and the middle ear, expose people to direct transmission of sars-cov-2 by inhalation or ocular projection of contaminated droplets, and to indirect transmission by contact with contaminated hands, objects or surfaces.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-29</td>\n",
       "      <td>Shetty, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/ors.12527\">10 “C” in COVID19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The routes of transmission include *contact of contaminated objects,*circulating droplets in the air called aerosols disseminated through *cough, sneeze, ocular secretions(2) from an infected individual.10 “c” in covid19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-30</td>\n",
       "      <td>Ye, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.jinf.2020.04.034\">Environmental Contamination of SARS-CoV-2 in Healthcare Premises</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The most contaminated objects were self-service printers (20.0%), desktop/keyboard (16.8%), and doorknob (16.0%).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Cai, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Indirect Virus Transmission in Cluster of COVID-19 Cases, Wenzhou, China, 2020</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Data indicated that indirect transmission of the causative virus occurred, perhaps resulting from virus contamination of common objects, virus aerosolization in a confined space, or spread from asymptomatic infected persons.indirect virus transmission in cluster of covid-19 cases, wenzhou, china, 2020.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-23</td>\n",
       "      <td>Castano, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Fomite transmission and disinfection strategies for SARS-CoV-2 and related viruses</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Contaminated objects or surfaces, referred to as fomites, play a critical role in the spread of viruses, including sars-cov-2, the virus responsible for the covid-19 pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Garcia-Alamino, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Aspectos Epidemiológicos, Clínica Y Mecanismos De Control De La Pandemia Por Sars-Cov-2: Situación En España./ [Epidemiological Aspects, Clinic And Control Mechanisms Of Sars-Cov-2pandemic: Situation In Spain]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Sars-cov-2 has a high transmission rate, the route of transmission between humans is through the secretions of infected people, hands or contaminated objects.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-04</td>\n",
       "      <td>Taylor, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.janxdis.2020.102232\">Development and Initial Validation of the COVID Stress Scales</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Abstract research and clinical observations suggest that during times of pandemic many people exhibit stress- or anxiety-related psychopathology that include fear of becoming infected, fear of coming into contact with possibly contaminated objects or surfaces, fear of foreigners who might be carrying infection (i.e., disease-related xenophobia), fear of the socio-economic consequences of the pandemic, compulsive checking and reassurance-seeking regarding possible pandemic-related threats, and traumatic stress symptoms about the pandemic (e.g., nightmares, intrusive thoughts).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Colaneri, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Severe acute respiratory syndrome coronavirus 2 RNA contamination of inanimate surfaces and virus viability in a health care emergency unit</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Objectives: to detect possible severe acute respiratory syndrome coronavirus-2 (sars-cov-2) rna contamination of inanimate surfaces in areas at high risk of aerosol formation by patients with coronavirus disease 2019 (covid-19).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Morvan, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Percutaneous dilatational tracheostomy for saturating influx of COVID-19 patients: Experience of military ENT physicians deployed in Mulhouse, France</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The secondary objective was to assess reliability and safety for patient and caregivers, with a risk of iatrogenic viral contamination.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>François, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">3D-printed contact-free devices designed and dispatched against the COVID-19 pandemic: The 3D COVID initiative</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Here, we report our experience in designing, producing and dispatching three 3d-printed objected intending to lower the risks of covid-19 contamination by limiting direct contacts: (1) fixed hand-free door openers, (2) door hooks and (3) button pushers.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-30</td>\n",
       "      <td>Aardema, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.jocrd.2020.100558\">COVID-19, Obsessive-Compulsive Disorder and Invisible Life Forms that Threaten the Self</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Specifically, it suggests that fears of contamination become obsessional when the threat of viruses and germs becomes personal, not solely in terms of its objective outcomes, but in how these life forms are able to threaten the self as the result of a fear of inner corruption characterizing those with ocd.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Malik, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Properties of Coronavirus and SARS-CoV-2</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Human-to-human transmission is primarily achieved through close contact of respiratory droplets, direct contact with the infected individuals, or by contact with contaminated objects and surfaces.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Abraham, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Using heat to kill SARS-CoV-2</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>One critically important issue is the proper sterilization of physical objects that have been contaminated by the virus.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-01</td>\n",
       "      <td>Malik, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Properties of Coronavirus and SARS-CoV-2.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Human-to-human transmission is primarily achieved through close contact of respiratory droplets, direct contact with the infected individuals, or by contact with contaminated objects and surfaces.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about stability of the virus in environmental conditions?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-04-14</td>\n",
       "      <td>Sun, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.09.20058875\">Stability of the COVID-19 virus under wet, dry and acidic conditions</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results showed that the covid-19 virus could survive for three days in wet and dry environments, but the dry condition is less favorable for the survival of the virus.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Sato, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Hyperglycemia enhances pancreatic cancer progression accompanied by elevations in phosphorylated STAT3 and MYC levels</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Preconditioning with a high- or low-glucose medium for 28 days showed that a high-glucose environment increased cell viability and sphere formation in panc-1, a kras-mutant human pancreatic ductal adenocarcinoma cell line, and mpkc1, a kras-mutant murine pancreatic cancer cell line.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Sato, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1371/journal.pone.0235573\">Hyperglycemia enhances pancreatic cancer progression accompanied by elevations in phosphorylated STAT3 and MYC levels.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Preconditioning with a high- or low-glucose medium for 28 days showed that a high-glucose environment increased cell viability and sphere formation in panc-1, a kras-mutant human pancreatic ductal adenocarcinoma cell line, and mpkc1, a kras-mutant murine pancreatic cancer cell line.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-14</td>\n",
       "      <td>Aboubakr, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/tbed.13707\">Stability of SARS‐CoV‐2 and other coronaviruses in the environment and on common touch surfaces and the influence of climatic conditions: A review</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>It has been suggested that large numbers of covid‐19 cases are associated with cold and dry climates in temperate regions of the world and that seasonality of the virus spread is suspected.stability of sars‐cov‐2 and other coronaviruses in the environment and on common touch surfaces and the influence of climatic conditions: a review.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Aboubakr, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Stability of SARS-CoV-2 and other coronaviruses in the environment and on common touch surfaces and the influence of climatic conditions: A review</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>It has been suggested that large numbers of covid-19 cases are associated with cold and dry climates in temperate regions of the world and that seasonality of the virus spread is suspected.stability of sars-cov-2 and other coronaviruses in the environment and on common touch surfaces and the influence of climatic conditions: a review.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-19</td>\n",
       "      <td>DiMaggio, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.14.20101691\">Blacks/African Americans are 5 Times More Likely to Develop COVID-19: Spatial Modeling of New York City ZIP Code-level Testing Results</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Areas with large proportions of black/african american residents are at markedly higher risk that is not fully explained by characteristics of the environment and pre-existing conditions in the population.blacks/african americans are 5 times more likely to develop covid-19: spatial modeling of new york city zip code-level testing results.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-01</td>\n",
       "      <td>Cuevas, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">SEI1I2HRSVM model applied to the coronavirus pandemic (COVID-19) in Paraguay</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In order to reflect the impact of the control measures adopted by the government and the population, the model employs variable transmission rates that change with the epidemiological status and environmental conditions.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-09</td>\n",
       "      <td>Freire-González, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1007/s10640-020-00448-7\">Pandemics and the Environmental Rebound Effect: Reflections from COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Although the pandemic offers potential to improve the environmental conditions, it brings also a high risk to produce jevons’ paradox, i.e., increase environmental burdens rather than decrease them, as initially expected.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Copiello, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">The spread of 2019-nCoV in China was primarily driven by population density. Comment on \"Association between short-term exposure to air pollution and COVID-19 infection: Evidence from China\" by Zhu et al</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>To this end, the relationship between demographic, socio-economic, and environmental conditions and the spread of the novel coronavirus in china is analyzed with spatial regression models on variables deflated by population size.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-26</td>\n",
       "      <td>Avhad, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.23.20111179\">On the COVID-19 Pandemic in Indian State of Maharashtra: Forecasting & Effect of different parameters</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Key findings of this work will be predominant in maintaining environment conditions at healthcare facilities to reduce transmission rates at these most vulnerable places.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-16</td>\n",
       "      <td>Copiello, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.scitotenv.2020.141028\">The spread of 2019-nCoV in China was primarily driven by population density. Comment on “Association between short-term exposure to air pollution and COVID-19 infection: Evidence from China” by Zhu et al.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>To this end, the relationship between demographic, socio-economic, and environmental conditions and the spread of the novel coronavirus in china is analyzed with spatial regression models on variables deflated by population size.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-09</td>\n",
       "      <td>Temmam, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.07.029090\">Absence of SARS-CoV-2 infection in cats and dogs in close contact with a cluster of COVID-19 patients in a veterinary campus</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>These original data can serve a better evaluation of the host range of sars-cov-2 in natural environment exposure conditions.absence of sars-cov-2 infection in cats and dogs in close contact with a cluster of covid-19 patients in a veterinary campus.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Harmooshi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Environmental concern regarding the effect of humidity and temperature on 2019-nCoV survival: fact or fiction</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Environmental conditions necessary for survival and spread of 2019-ncov are somewhat transparent but unlike animal coronaviruses.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Lokhandwala, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Indirect impact of COVID-19 on environment: A brief study in Indian context</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Reports from all over the world are indicating that after the outbreak of covid-19, environmental conditions including air quality and water quality in rivers are improving and wildlife is blooming.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-18</td>\n",
       "      <td>Lokhandwala, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.envres.2020.109807\">Indirect impact of COVID-19 on Environment: A brief study in Indian Context</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Reports from all over the world are indicating that after the outbreak of covid-19, environmental conditions including air quality and water quality in rivers are improving and wildlife is blooming.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about persistence of the virus on various substrates? (e.g., nasal discharge, sputum, urine, fecal matter, blood)</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-03-26</td>\n",
       "      <td>Tan, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.24.20042382\">Viral Kinetics and Antibody Responses in Patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Clinical specimens including nasopharyngeal swab, sputum, blood, urine and stool were tested periodically according to standardized case report form with final follow-up on february 27.<br><br>Conclusions nasopharyngeal, sputum and stools rather than blood and urine, were the major shedding routes for sars-cov-2, and meanwhile sputum had a prolonged viral shedding.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-08</td>\n",
       "      <td>Woelfel, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.05.20030502\">Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods: we studied viral load courses by rt-pcr in oro- and nasopharyngeal swabs, sputum, stool, blood, and urine in nine hospitalized cases.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about persistence of the virus on various surfaces? (e,g., copper, stainless steel, plastic) </h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Ren, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.12998/wjcc.v8.i8.1391\">Stability and infectivity of coronaviruses in inanimate environments</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This review aims to summarize data on the persistence of different coronaviruses on inanimate surfaces.<br><br>All reports with experimental evidence on the duration persistence of coronaviruses on any type of surface were included.<br><br>Most viruses from the respiratory tract, such as coronaviruses, influenza, sars-cov, or rhinovirus, can persist on surfaces for a few days.<br><br>Persistence time on inanimate surfaces varied from minutes to up to one month, depending on the environmental conditions.<br><br>The most common coronaviruses may well survive or persist on surfaces for up to one month.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-26</td>\n",
       "      <td>Whitworth, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1128/aem.01482-20\">Persistence of Bacteriophage Phi 6 on Porous and Non-Porous Surfaces; Potential for use as Ebola or Coronavirus Surrogate.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The persistence of phi 6 was evaluated as a surrogate for ebov and coronaviruses on porous and nonporous hospital surfaces.<br><br>These data convey to public health investigators that enveloped viruses can persist and remain infective on surfaces, thus demonstrating a potential risk for transmission.<br><br>Under these laboratory-simulated western indoor hospital conditions, phi 6 was used to assess suitability as a surrogate for environmental persistence research related to enveloped viruses, including ebov and coronaviruses.persistence of bacteriophage phi 6 on porous and non-porous surfaces; potential for use as ebola or coronavirus surrogate.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Whitworth, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Persistence of Bacteriophage Phi 6 on Porous and Non-Porous Surfaces; Potential for use as Ebola or Coronavirus Surrogate</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The persistence of phi 6 was evaluated as a surrogate for ebov and coronaviruses on porous and nonporous hospital surfaces.<br><br>These data convey to public health investigators that enveloped viruses can persist and remain infective on surfaces, thus demonstrating a potential risk for transmission.<br><br>Under these laboratory-simulated western indoor hospital conditions, phi 6 was used to assess suitability as a surrogate for environmental persistence research related to enveloped viruses, including ebov and coronaviruses.persistence of bacteriophage phi 6 on porous and non-porous surfaces; potential for use as ebola or coronavirus surrogate.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-05</td>\n",
       "      <td>Akram, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1007/s13337-020-00603-0\">Inanimate surfaces as potential source of 2019-nCoV spread and their disinfection with biocidal agents</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Human coronaviruses can persist on inanimate surfaces such as plastic, glass, fibers and metals up to nine days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-30</td>\n",
       "      <td>Kampf, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.infpip.2020.100044\">Potential role of inanimate surfaces for the spread of coronaviruses and their inactivation with disinfectant agents</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In a recent review on the persistence of human and veterinary coronaviruses on inanimate surfaces it was shown that human coronaviruses such as severe acute respiratory syndrome (sars) coronavirus, middle east respiratory syndrome (mers) coronavirus or endemic human coronaviruses (hcov) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Kampf, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Persistence of coronaviruses on inanimate surfaces and their inactivation with biocidal agents</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We therefore reviewed the literature on all available information about the persistence of human and veterinary coronaviruses on inanimate surfaces as well as inactivation strategies with biocidal agents used for chemical disinfection, e.g.<br><br>The analysis of 22 studies reveals that human coronaviruses such as severe acute respiratory syndrome (sars) coronavirus, middle east respiratory syndrome (mers) coronavirus or endemic human coronaviruses (hcov) can persist on inanimate surfaces like metal, glass or plastic for up to 9 days, but can be efficiently inactivated by surface disinfection procedures with 62-71% ethanol, 0.5% hydrogen peroxide or 0.1% sodium hypochlorite within 1 minute.<br><br>As no specific therapies are available for sars-cov-2, early containment and prevention of further spread will be crucial to stop the ongoing outbreak and to control this novel infectious thread.persistence of coronaviruses on inanimate surfaces and their inactivation with biocidal agents.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-12</td>\n",
       "      <td>Kasloff, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.11.20128884\">Stability of SARS-CoV-2 on Critical Personal Protective Equipment</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Although sars-cov-2 virus has been found to persist on surfaces for a number of days, the extent and duration of fomites as a mode of transmission, particularly in healthcare settings, has not been fully characterized.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Biryukov, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Increasing Temperature and Relative Humidity Accelerates Inactivation of SARS-CoV-2 on Surfaces</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>These findings suggest that a potential for fomite transmission may persist for hours to days in indoor environments and have implications for assessment of the risk posed by surface contamination in indoor environments.importance mitigating the transmission of sars-cov-2 in clinical settings and public spaces is critically important to reduce the number of covid-19 cases while effective vaccines and therapeutics are under development.<br><br>In this context, understanding the factors contributing to sars-cov-2 persistence on surfaces will enable a more accurate estimation of the risk of contact transmission and inform mitigation strategies.<br><br>To this end, we have developed a simple mathematical model that can be used to estimate virus decay on nonporous surfaces under a range of conditions and which may be utilized operationally to identify indoor environments in which the virus is most persistent.increasing temperature and relative humidity accelerates inactivation of sars-cov-2 on surfaces.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-26</td>\n",
       "      <td>Dave, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.24.20139469\">Characterization of a novel, low-cost, scalable ozone gas system for sterilization of N95 respirators and other COVID-19 related use cases.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>New studies investigating the tenacity of sars-cov-2 have highlighted its ability to persist on a myriad of surfaces for several days, including gowns and shoes.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Shin, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Environmental surface testing for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) during prolonged isolation of an asymptomatic carrier</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Environmental surface testing was performed to search for evidence of severe acute respiratory coronavirus virus 2 (sars-cov-2) environmental contamination by an asymptomatic sars-cov-2 carrier with persistently high viral loads under isolation.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Gerlier, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Persistance du SARS-CoV-2 infectieux sur les surfaces inertes et transmission par les mains./ Persistence of infectious SARS-CoV-2 on inert surfaces and hand-mediated transmission</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>No data providedpersistance du sars-cov-2 infectieux sur les surfaces inertes et transmission par les mains./ persistence of infectious sars-cov-2 on inert surfaces and hand-mediated transmission.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-20</td>\n",
       "      <td>Pezzotti, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.19.159970\">Rapid Inactivation of SARS-CoV-2 by Silicon Nitride, Copper, and Aluminum Nitride</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Further studies are needed to validate these findings and investigate whether silicon nitride can be incorporated into personal protective equipment and commonly touched surfaces, as a strategy to discourage viral persistence and disease spread.rapid inactivation of sars-cov-2 by silicon nitride, copper, and aluminum nitride.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-01</td>\n",
       "      <td>Gerlier, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1684/vir.2020.0849\">Persistence of infectious SARS-CoV-2 on inert surfaces and hand-mediated transmission.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>No data providedpersistence of infectious sars-cov-2 on inert surfaces and hand-mediated transmission.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Ratnesar-Shumate, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Simulated Sunlight Rapidly Inactivates SARS-CoV-2 on Surfaces</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The present study provides the first evidence that sunlight may rapidly inactivate sars-cov-2 on surfaces, suggesting that persistence, and subsequently exposure risk, may vary significantly between indoor and outdoor environments.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-03</td>\n",
       "      <td>Contreras, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.chaos.2020.110087\">Statistically-based methodology for revealing real contagion trends and correcting delay-induced errors in the assessment of COVID-19 pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Particularities of sars-cov-2, such as its persistence in surfaces and the lack of a curative treatment or vaccine against covid-19, have pushed authorities to apply restrictive policies to control its spreading.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about viral shedding duration?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-05-23</td>\n",
       "      <td>Weiss, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.21.20108605\">Spatial and temporal dynamics of SARS-CoV-2 in COVID-19 patients: A systematic review</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In this study, we aimed to provide a coherent overview from published studies of the duration of viral detection and viral load in covid-19 patients, stratified by specimen type, clinical severity and age.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Dodds, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Model-Informed Drug Repurposing: Viral Kinetic Modeling to Prioritize Rational Drug Combinations for COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The endpoints and metrics included viral load area under the curve (auc), duration of viral shedding, and epithelial cells infected.<br><br>In addition, we observed that the time-window opportunity for a therapeutic intervention to effect duration of viral shedding exceeds the effect on sparing epithelial cells from infection or impact on viral load auc.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-21</td>\n",
       "      <td>Dodds, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/bcp.14486\">Model-Informed Drug Repurposing: Viral Kinetic Modeling to Prioritize Rational Drug Combinations for COVID-19.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The endpoints and metrics included viral load area under the curve (auc), duration of viral shedding, and epithelial cells infected.<br><br>In addition, we observed that the time-window opportunity for a therapeutic intervention to effect duration of viral shedding exceeds the effect on sparing epithelial cells from infection or impact on viral load auc.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-28</td>\n",
       "      <td>Cevik, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.07.25.20162107\">SARS-CoV-2 viral load dynamics, duration of viral shedding and infectiousness: a living systematic review and meta-analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Background viral load kinetics and the duration of viral shedding are important determinants for disease transmission.<br><br>We aim i) to characterise viral load dynamics, duration of viral rna, and viable virus shedding of sars-cov-2 in various body fluids and ii) to compare sars-cov-2 viral dynamics with sars-cov-1 and mers-cov.<br><br>Methods: medline, embase, europe pmc, preprint servers and grey literature were searched to retrieve all articles reporting viral dynamics and duration of sars-cov-2, sars-cov-1 and mers-cov shedding.<br><br>Funding: no funding was received.sars-cov-2 viral load dynamics, duration of viral shedding and infectiousness: a living systematic review and meta-analysis.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Warabi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Effects of oral care on prolonged viral shedding in coronavirus disease 2019 (COVID-19)</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods and results: we evaluated the clinical course of eight covid-19 patients, including their duration of viral shedding, by pcr testing of nasopharyngeal swabs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-24</td>\n",
       "      <td>Warabi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/scd.12498\">Effects of oral care on prolonged viral shedding in coronavirus disease 2019 (COVID-19).</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods and results we evaluated the clinical course of eight covid-19 patients, including their duration of viral shedding, by pcr testing of nasopharyngeal swabs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Chen, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Associations of Clinical Characteristics and Treatment Regimens with Viral RNA Shedding Duration in Patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: the median viral rna shedding duration was 12 days (interquartile range, 8-16 d) after the onset of illness.<br><br>Lopinavir/ritonavir use may be associated with prolonged viral rna shedding in non-severe patients; further randomized controlled trials are needed to confirm this finding.associations of clinical characteristics and treatment regimens with viral rna shedding duration in patients with covid-19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-26</td>\n",
       "      <td>Tan, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.24.20042382\">Viral Kinetics and Antibody Responses in Patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The routes and duration of viral shedding, antibody response, and their associations with disease severity and clinical manifestations were systematically evaluated.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Duration of SARS-CoV-2 RNA shedding and factors associated with prolonged viral shedding in patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A retrospective cohort of covid-19 patients admitted to a designated hospital in beijing was analyzed to study the factors affecting the duration of viral shedding.<br><br>The median duration of viral shedding was 11 days (iqr, 8-14.3 days) as measured from illness onset.<br><br>Univariate regression analysis showed that disease severity, corticosteroid therapy, fever (temperature>38.5°c), and time from onset to hospitalization were associated with prolonged duration of viral shedding (p < .05).<br><br>Multivariate regression analysis showed that fever (temperature>38.5°c) (or, 5.1, 95%ci: 1.5-18.1), corticosteroid therapy (or, 6.3, 95%ci: 1.5-27.8), and time from onset to hospitalization (or, 1.8, 95%ci: 1.19-2.7) were associated with increased odds of prolonged duration of viral shedding.<br><br>Corticosteroid treatment, fever (temperature>38.5°c), and longer time from onset to hospitalization were associated with prolonged viral shedding in covid-19 patients.duration of sars-cov-2 rna shedding and factors associated with prolonged viral shedding in patients with covid-19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-09</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1002/jmv.26280\">Duration of SARS‐CoV‐2 RNA shedding and factors associated with prolonged viral shedding in patients with COVID‐19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods: a retrospective cohort of covid‐19 patients admitted to a designated hospital in beijing was analyzed to study the factors affecting the duration of viral shedding.<br><br>Results: the median duration of viral shedding was 11 days (iqr, 8‐14.3 days) as measured from illness onset.<br><br>Univariate regression analysis showed that disease severity, corticosteroid therapy, fever (temperature>38.5℃), and time from onset to hospitalization were associated with prolonged duration of viral shedding (p<0.05).<br><br>Multivariate regression analysis showed that fever (temperature>38.5℃) (or 5.1, 95%ci: 1.5‐18.1), corticosteroid therapy (or 6.3, 95%ci: 1.5‐27.8), and time from onset to hospitalization (or 1.8, 95%ci: 1.19‐2.7) were associated with increased odds of prolonged duration of viral shedding.<br><br>All rights reserved.duration of sars‐cov‐2 rna shedding and factors associated with prolonged viral shedding in patients with covid‐19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Zhou, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Viral dynamics in asymptomatic patients with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>However, the duration of viral shedding remained similar in the two groups (7 days, iqr 5-14 days vs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-14</td>\n",
       "      <td>Goyal, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.10.20061325\">Potency and timing of antiviral therapy as determinants of duration of SARS CoV-2 shedding and intensity of inflammatory response</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Potent therapies dosed soon after peak viral load when infected people typically develop symptoms, are predicted to decrease shedding duration and intensity of the effector immune response, but to have little effect on viral area under the curve, which is driven by high levels of early sars cov-2 replication.<br><br>Our results support an early test and treat approach for covid-19, but also demonstrate the need to identify early viral shedding kinetic features that are the most predictive surrogates of clinical severity and transmission risk.potency and timing of antiviral therapy as determinants of duration of sars cov-2 shedding and intensity of inflammatory response.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Chan, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">A systematic review on COVID-19: urological manifestations, viral RNA detection and special considerations in urological conditions</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>5.74% (95% ci 2.88-9.44%) of covid-19 patients had positive viral rna in urine samples, but the duration of viral shedding in urine was unknown.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-30</td>\n",
       "      <td>Xu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1097/inf.0000000000002814\">Duration of Respiratory and Gastrointestinal Viral Shedding in Children With SARS-CoV-2: A Systematic Review and Synthesis of Data.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Little is known about the duration of respiratory and gastrointestinal viral shedding in children with covid-19.<br><br>Methods we performed a systematic search of ovid medline, embase and cochrane central databases for studies reporting real-time reverse transcriptase polymerase chain reaction (rt-pcr) results in children with covid-19, then extracted and synthesized data on duration of viral shedding from symptom onset in respiratory and gastrointestinal samples.<br><br>Results based on data compiled from 69 pediatric cases, the duration of viral shedding through the respiratory tract is up to 24 days from symptom onset with a mean of 11.1 ± 5.8 days.<br><br>The mean duration of viral shedding via the gastrointestinal tract was 23.6 ± 8.8 days from symptom onset.<br><br>Conclusions to our knowledge, this is the first attempt to systematically review the duration of respiratory and gastrointestinal viral shedding of sars-cov-2 in pediatric patients.<br><br>These findings may have important implications for infection control strategies during the covid-19 pandemic.duration of respiratory and gastrointestinal viral shedding in children with sars-cov-2: a systematic review and synthesis of data.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Xu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Duration of Respiratory and Gastrointestinal Viral Shedding in Children With SARS-CoV-2: A Systematic Review and Synthesis of Data</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Little is known about the duration of respiratory and gastrointestinal viral shedding in children with covid-19.<br><br>Methods: we performed a systematic search of ovid medline, embase and cochrane central databases for studies reporting real-time reverse transcriptase polymerase chain reaction (rt-pcr) results in children with covid-19, then extracted and synthesized data on duration of viral shedding from symptom onset in respiratory and gastrointestinal samples.<br><br>Results: based on data compiled from 69 pediatric cases, the duration of viral shedding through the respiratory tract is up to 24 days from symptom onset with a mean of 11.1 ± 5.8 days.<br><br>The mean duration of viral shedding via the gastrointestinal tract was 23.6 ± 8.8 days from symptom onset.<br><br>Conclusions: to our knowledge, this is the first attempt to systematically review the duration of respiratory and gastrointestinal viral shedding of sars-cov-2 in pediatric patients.<br><br>These findings may have important implications for infection control strategies during the covid-19 pandemic.duration of respiratory and gastrointestinal viral shedding in children with sars-cov-2: a systematic review and synthesis of data.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about viral shedding in fecal/stool?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Ma, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Do children need a longer time to shed SARS-CoV-2 in stool than adults?</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Sars-cov-2 can be shed in the stool of patients in the recovery phase.<br><br>We analyzed the possible causes of this finding and recommend that a negative stool sample be included in a patient's discharge criteria.do children need a longer time to shed sars-cov-2 in stool than adults?.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Cheung, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Gastrointestinal Manifestations of SARS-CoV-2 Infection and Virus Load in Fecal Samples From a Hong Kong Cohort: Systematic Review and Meta-analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We performed a systematic review and meta-analysis of published gastrointestinal symptoms and detection of virus in stool and also summarized data from a cohort of patients with covid-19 in hong kong.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-03</td>\n",
       "      <td>Cheung, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1053/j.gastro.2020.03.065\">Gastrointestinal Manifestations of SARS-CoV-2 Infection and Virus Load in Fecal Samples from the Hong Kong Cohort and Systematic Review and Meta-analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We performed a systematic review and meta-analysis of published gastrointestinal symptoms and detection of virus in stool, and also summarized data from a cohort of patients with covid-19 in hong kong.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-24</td>\n",
       "      <td>Olesen, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.24.169094\">Modelling donor screening strategies to reduce the risk of SARS-CoV-2 via fecal microbiota transplantation</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The potential for transmission of sars-cov-2 shed in stool via fecal microbiota transplantation is not yet known, and the effectiveness of various testing strategies to prevent fmt-based transmission has also not yet been quantified.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-23</td>\n",
       "      <td>De et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1093/jpids/piaa065\">Dynamic viral SARS-CoV-2 RNA shedding in in children: preliminary data and clinical consideration of Italian regional center.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>These data underline the significance of nasopharyngeal and stool swab for detecting infected children; further studies are needed for transmissibility.dynamic viral sars-cov-2 rna shedding in in children: preliminary data and clinical consideration of italian regional center.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Lo, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Evaluation of SARS-CoV-2 RNA shedding in clinical specimens and clinical characteristics of 10 patients with COVID-19 in Macau</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Based on our data on sars-cov-2 rna shedding in stool and the possibility of a lag in viral detection in nps specimens, the assessment of both fecal and respiratory specimen is recommended to enhance diagnostic sensitivity, and also to aid discharge decision before the role of viral rna shedding in stool is clarified.evaluation of sars-cov-2 rna shedding in clinical specimens and clinical characteristics of 10 patients with covid-19 in macau.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-28</td>\n",
       "      <td>Cevik, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.07.25.20162107\">SARS-CoV-2 viral load dynamics, duration of viral shedding and infectiousness: a living systematic review and meta-analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Mean sars-cov-2 rna shedding duration in upper respiratory tract, lower respiratory tract, stool and serum were 17.0, 14.6, 17.2 and 16.6 days, respectively.<br><br>Maximum duration of sars-cov-2 rna shedding reported in urt, lrt, stool and serum was 83, 59, 35 and 60 days, respectively.<br><br>Conclusion: although sars-cov-2 rna shedding in respiratory and stool can be prolonged, duration of viable virus is relatively short-lived.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-09</td>\n",
       "      <td>Lee, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.08.033001\">Relative Abundance of SARS-CoV-2 Entry Genes in the Enterocytes of the Lower Gastrointestinal Tract</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>While covid-19 frequently manifests as a respiratory infection,1 there is evidence for infection of the gastrointestinal (gi) tract1–4 with documented viral rna shedding in the stool of infected patients.2,4 in this study, we aimed to investigate the expression of ace2 and tmprss2, which are required for sars-cov-2 entry into mammalian cells,5 from single-cell rna sequencing (scrna-seq) datasets of five different parts of the gi tract: esophagus, stomach, pancreas, small intestine, and colon/rectum.relative abundance of sars-cov-2 entry genes in the enterocytes of the lower gastrointestinal tract.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Liu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Positive SARS-CoV-2 RNA recurs repeatedly in a case recovered from COVID-19: dynamic results from 108 days of follow-up</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In this case, positive sars-cov-2 recurred even after apparent recovery (normal ct imaging, no clinical symptoms, negative sars-cov-2 on stool sample and negative serum igm test) from covid-19, viral shedding duration lasted for 65 days, the time from symptom onset to disappearance was up to 95 days.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-12-01</td>\n",
       "      <td>Liu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1080/22221751.2020.1772677\">Dynamic surveillance of SARS-CoV-2 shedding and neutralizing antibody in children with COVID-19.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median duration of viral shedding detected in nasopharyngeal swabs, oropharyngeal swabs, and stools was 13, 4, and 43 days respectively, and the maximum duration of viral shedding detected from stools was 46 days after discharge.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Liu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Dynamic surveillance of SARS-CoV-2 shedding and neutralizing antibody in children with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The median duration of viral shedding detected in nasopharyngeal swabs, oropharyngeal swabs, and stools was 13, 4, and 43 days respectively, and the maximum duration of viral shedding detected from stools was 46 days after discharge.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-09</td>\n",
       "      <td>Odih, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.4269/ajtmh.20-0462\">Could Water and Sanitation Shortfalls Exacerbate SARS-CoV-2 Transmission Risks?</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Sars-cov-2, the etiologic agent of covid-19, is shed in stool.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>D039,; et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Diarrhea During COVID-19 Infection: Pathogenesis, Epidemiology, Prevention, and Management</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Several reports indicate viral rna shedding in stool detectable longer time period than in nasopharyngeal swabs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-08</td>\n",
       "      <td>D’Amico, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.cgh.2020.04.001\">Diarrhea during COVID-19 infection: pathogenesis, epidemiology, prevention and management</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Several reports indicate viral rna shedding in stool detectable longer time period than in nasopharyngeal swabs.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-08</td>\n",
       "      <td>Woelfel, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.05.20030502\">Clinical presentation and virological assessment of hospitalized cases of coronavirus disease 2019 in a travel-associated transmission cluster</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Sars-cov-2 can actively replicate in the upper respiratory tract, and is shed for a prolonged time after symptoms end, including in stool.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about viral shedding from nasopharynx?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-06-30</td>\n",
       "      <td>L'Huillier, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.3201/eid2610.202403\">Culture-Competent SARS-CoV-2 in Nasopharynx of Symptomatic Neonates, Children, and Adolescents.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our findings show that symptomatic neonates, children, and teenagers shed infectious sars-cov-2, suggesting that transmission from them is plausible.culture-competent sars-cov-2 in nasopharynx of symptomatic neonates, children, and adolescents.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>L039,; et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Culture-Competent SARS-CoV-2 in Nasopharynx of Symptomatic Neonates, Children, and Adolescents</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our findings show that symptomatic neonates, children, and teenagers shed infectious sars-cov-2, suggesting that transmission from them is plausible.culture-competent sars-cov-2 in nasopharynx of symptomatic neonates, children, and adolescents.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-03</td>\n",
       "      <td>Young, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1001/jama.2020.3204\">Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2 in Singapore.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results among the 18 hospitalized patients with pcr-confirmed sars-cov-2 infection (median age, 47 years; 9 [50%] women), clinical presentation was an upper respiratory tract infection in 12 (67%), and viral shedding from the nasopharynx was prolonged for 7 days or longer among 15 (83%).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Young, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2 in Singapore</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Results: among the 18 hospitalized patients with pcr-confirmed sars-cov-2 infection (median age, 47 years; 9 [50%] women), clinical presentation was an upper respiratory tract infection in 12 (67%), and viral shedding from the nasopharynx was prolonged for 7 days or longer among 15 (83%).</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about viral shedding in blood?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020-04-17</td>\n",
       "      <td>Zeng, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.15.20063107\">Association between ABO blood groups and clinical outcome of coronavirus disease 2019: Evidence from two cohorts</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>With 265 patients from multiple medical centers and two established cohorts, we found that the blood type a population was more sensitive to sars-cov-2.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Baron, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Patient blood management during the COVID-19 pandemic: a narrative review</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The aim of this expert review is to provide clinicians and healthcare authorities with information regarding how to apply established principles of patient blood management during the covid-19 pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-27</td>\n",
       "      <td>Baron, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/anae.15095\">Patient blood management during the COVID-19 pandemic - a narrative review.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The aim of this expert review is to provide clinicians and health care authorities with information regarding how to apply established principles of patient blood management during the covid-19 pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-28</td>\n",
       "      <td>Epstein, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/vox.12964\">Use of COVID‐19 convalescent plasma in low‐ and middle‐income countries: a call for ethical principles and the assurance of quality and safety</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The working party on global blood safety of the international society of blood transfusion has published “points to consider in the preparation and transfusion of covid‐19 convalescent plasma in low‐ and middle‐ income countries”.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Zeng, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Can we predict the severity of coronavirus disease 2019 with a routine blood test?</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Methods: pubmed, embase, cochrane library, wanfang, and china national knowledge infrastructure (cnki) databases were searched to identify studies reporting data on markers in the routine blood test and the severity of covid­19, published until march 20, 2020.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-08</td>\n",
       "      <td>Gérard, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/bjh.16884\">COVID‐19 and ABO blood group: another viewpoint</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>[1] have recently published \"an association between abo blood groups and risk of sars-cov-2 pneumonia\", an observation already reported a few weeks ago as a medrxiv preprint by jiao zhao et al.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-09</td>\n",
       "      <td>Lam, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/bjh.16721\">Management of COVID‐19‐related paediatric blood samples in a clinical haematology laboratory</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The first study to be published on the initial 41 cases of covid-19 infections admitted in wuhan detected sars-cov-2 rna in the blood of 6/41 (15%) of patients (huang et.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-02</td>\n",
       "      <td>Roxhed, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.07.01.20143966\">A translational multiplex serology approach to profile the prevalence of anti-SARS-CoV-2 antibodies in home-sampled blood</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We established a translational approach combining home blood sampling by finger-pricking with multiplexed serology to assess the exposure to the sars-cov-2 virus in a general population.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-17</td>\n",
       "      <td>Deng, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1515/cclm-2020-0338\">Blood biochemical characteristics of patients with coronavirus disease 2019 (COVID-19): a systemic review and meta-analysis.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This study aimed to summarize the blood biochemistry characteristics of covid-19 patients by performing a systemic review and meta-analysis of published studies.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Deng, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Blood biochemical characteristics of patients with coronavirus disease 2019 (COVID-19): a systemic review and meta-analysis</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This study aimed to summarize the blood biochemistry characteristics of covid-19 patients by performing a systemic review and meta-analysis of published studies.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Zhang, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We found the presence of 2019-ncov in anal swabs and blood as well, and more anal swab positives than oral swab positives in a later stage of infection, suggesting shedding and thereby transmitted through oral-fecal route.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Sanchis-Gomar, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Angiotensin-Converting Enzyme 2 and Antihypertensives (Angiotensin Receptor Blockers and Angiotensin-Converting Enzyme Inhibitors) in Coronavirus Disease 2019</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Several professional health organizations have published position papers stating that there is no evidence to change the use of angiotensin-converting enzyme inhibitors (aceis) or angiotensin receptor blockers (arbs) in the management of elevated blood pressure in the context of avoiding or treating covid-19 infection.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Kalafat, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Lung ultrasound and computed tomographic findings in pregnant woman with COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The neonatal swabs, cord blood and placental swab rt-pcr tests were negative for sars-cov-2, a finding consistent with the published literature suggesting no vertical transmission of this virus in pregnant women.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Hughes, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Predictions of Systemic, Intracellular, and Lung Concentrations of Azithromycin With Different Dosing Regimens Used in COVID-19 Clinical Trials</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A population pharmacokinetic model was implemented to predict lung, intracellular poly/mononuclear cell (peripheral blood monocyte (pbm)/polymorphonuclear leukocyte (pml)), and alveolar macrophage (am) concentrations using published data and compared against preclinical effective concentration 90% (ec90 ) for severe acute respiratory syndrome-coronavirus 2 (sars-cov-2).</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Shen, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Epidemiology and clinical course of COVID-19 in Shanghai, China</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Patients with mild disease using glucocorticoid tended to have longer viral shedding in blood and feces.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about viral shedding in urine?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Funk, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Smothering fecal-oral coronavirus spread</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A review the research of zang et al (sci immunol , 2020, 5, 3582) is reviewed diarrhea is a common symptom in patients with coronavirus disease 2019 (covid-19) zang et al used organoid cultures of cells from the epithelial lining of human small and large intestine as an in vitro model system with which to study viral entry and replication in enterocytes mature enterocytes expressing the viral receptor were susceptible to productive infection, which was also stimulated by the expression of a protease involved in viral entry a subset of patients with covid-19 shed high amounts of viral rna in feces, but experiments with simulated human colonic fluid suggested that any shed virus would be rapidly inactivated during transit through the colonsmothering fecal-oral coronavirus spread.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-29</td>\n",
       "      <td>Schnapp, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/jdv.16749\">Introductory histopathological findings may shed light on COVID‐19 paediatric hyperinflammatory shock syndrome</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Recent report from italy reported a 30 time increase in the rate of kawasaki-like presentation during the covid-19 pandemic among children, in many cases the nasopharyngeal swabs taken from these children were negative for covid-19, and the association with covid-19 infection is unclear.introductory histopathological findings may shed light on covid‐19 paediatric hyperinflammatory shock syndrome.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-12</td>\n",
       "      <td>Farkas, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.coesh.2020.06.001\">Wastewater and public health: the potential of wastewater surveillance for monitoring COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>As the amount of virus shed in faeces and urine varies largely from person to person, it is very difficult to quantitatively determine the number of people who are infected in the population.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-03-26</td>\n",
       "      <td>Santarpia, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.03.23.20039446\">Transmission Potential of SARS-CoV-2 in Viral Shedding Observed at the University of Nebraska Medical Center</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>During the initial isolation of 13 individuals confirmed positive with covid-19 infection, air and surface samples were collected in eleven isolation rooms to examine viral shedding from isolated individuals.<br><br>Many commonly used items, toilet facilities, and air samples had evidence of viral contamination, indicating that sars-cov-2 is shed to the environment as expired particles, during toileting, and through contact with fomites.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-01</td>\n",
       "      <td>Lu, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.07.01.182659\">High affinity binding of SARS-CoV-2 spike protein enhances ACE2 carboxypeptidase activity</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>These results highlighted the altered activity of ace2 during sars-cov-2 infection and would shed new lights on the pathogenesis of covid-19 and its complications for better treatments.high affinity binding of sars-cov-2 spike protein enhances ace2 carboxypeptidase activity.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-10</td>\n",
       "      <td>Bhatia, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.4103/aian.aian_380_20\">COVID-19 and Stroke: Incidental, Triggered or Causative</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Data from stroke patients gathered in large multicentric cohorts could help shed more light on the occurrence, behaviour, aetiology, pathophysiology, biomarkers and outcomes of stroke occurring during the ongoing pandemic.covid-19 and stroke: incidental, triggered or causative.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>DA et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">COVID-19 and Health-Related Authority Allocation Puzzles</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This leads to puzzles concerning (a) the principles for justifiably allocating \"powers\" in these domains and (b) whether and how they change during \"emergencies.\" this work motivates the puzzles, explains why resolving them should be part of long-term responses to covid-19, and outlines some initial covid-19-related findings that shed light on justifiable authority allocation, emergencies, emergency powers, and the relationships between them.covid-19 and health-related authority allocation puzzles.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Kim, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Infection and Rapid Transmission of SARS-CoV-2 in Ferrets</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Although fatalities were not observed, sars-cov-2-infected ferrets shed virus in nasal washes, saliva, urine, and feces up to 8 days post-infection.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-25</td>\n",
       "      <td>Pikoulis, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.puhe.2020.05.036\">Gathering evidence on the decreased emergency room visits during theCovid 19 pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Investigating the causes of the decline in er visits is of outmost public health relevance as it might shed light in the indirect health effects of covid-19 pandemic.gathering evidence on the decreased emergency room visits during thecovid 19 pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-21</td>\n",
       "      <td>Leng, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Analysis of misinformation during the COVID-19 outbreak in China: cultural, social and political entanglements</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Our findings shed light on the distinct characteristics of misinformation during the covid-19 and offer insights into combating misinformation in china and across the world at large.analysis of misinformation during the covid-19 outbreak in china: cultural, social and political entanglements.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-10-01</td>\n",
       "      <td>Miripour, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.bios.2020.112435\">Real-time diagnosis of reactive oxygen species (ROS) in fresh sputum by electrochemical tracing; correlation between COVID-19 and viral-induced ROS in lung/respiratory epithelium during this pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Testing the sensor in more than 4 hospitals shed promising lights in ros based real-time tracing of covid-19 from the sputum sample.real-time diagnosis of reactive oxygen species (ros) in fresh sputum by electrochemical tracing; correlation between covid-19 and viral-induced ros in lung/respiratory epithelium during this pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-21</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.05.18.20105155\">The Dynamic Changes of Antibodies against SARS-CoV-2 during the Infection and Recovery of COVID-19</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Therefore, our evidence proves that the dynamic changes of antibodies could provide an important reference for diagnosis, monitoring, and treatment, and shed new light on the precise management of covid-19.the dynamic changes of antibodies against sars-cov-2 during the infection and recovery of covid-19.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-13</td>\n",
       "      <td>Miotto, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Does blood type affect the COVID-19 infection pattern?</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>(2020) presented a hypothesis that could shed some light on i) and ii), specifically, it has been proposed that during the subject-to-subject transfer sars-cov-2 conserves on its capsid the erythrocytes' antigens of the source subject, causing a potential immune reaction in a receiving subject which has previously acquired specific antibodies to the source subject antigens.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-05</td>\n",
       "      <td>South, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1161/hypertensionaha.120.15291\">ACE2, COVID-19, and ACE Inhibitor and ARB Use during the Pandemic: The Pediatric Perspective.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Given the importance of the angiotensin-converting enzyme 2/angiotensin-(1-7) pathway and the potential differences between adults and children, it is crucial that children are included in coronavirus-related research, as this may shed light on potential mechanisms for why children are at decreased risk of severe coronavirus disease 2019.ace2, covid-19, and ace inhibitor and arb use during the pandemic: the pediatric perspective.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>South, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">ACE2 (Angiotensin-Converting Enzyme 2), COVID-19, and ACE Inhibitor and Ang II (Angiotensin II) Receptor Blocker Use During the Pandemic: The Pediatric Perspective</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Given the importance of the ace 2/ang-(1-7) pathway and the potential differences between adults and children, it is crucial that children are included in coronavirus-related research, as this may shed light on potential mechanisms for why children are at decreased risk of severe covid-19.ace2 (angiotensin-converting enzyme 2), covid-19, and ace inhibitor and ang ii (angiotensin ii) receptor blocker use during the pandemic: the pediatric perspective.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<h3>Q: What do we know about implemtation of diagnostics?</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>pub_date</th>\n",
       "      <th>authors</th>\n",
       "      <th>title</th>\n",
       "      <th>excerpt</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>McDermott, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Utilising Point of Care Diagnostics to Minimise Nosocomial Infection in the 2019 Novel Coronavirus (SARS-CoV-2) Pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>No data providedutilising point of care diagnostics to minimise nosocomial infection in the 2019 novel coronavirus (sars-cov-2) pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-03</td>\n",
       "      <td>McDonald, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1111/acem.14039\">Diagnostic Performance of a Rapid Point of Care Test for SARS‐CoV‐2 in an Urban ED Setting</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>This was made possible by increasing availability of diagnostic molecular tests in several formats, from laboratory based traditional, rt‐pcr methods to near patient testing rapid point of care pcr tests.diagnostic performance of a rapid point of care test for sars‐cov‐2 in an urban ed setting.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-04</td>\n",
       "      <td>Osterdahl, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.01.20047357\">Detecting SARS-CoV-2 at point of care: Preliminary data comparing Loop-mediated isothermal amplification (LAMP) to PCR</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A cost effective and efficient diagnostic tool as near to the point of care (poc) as possible would be a game changer in current testing.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-18</td>\n",
       "      <td>Mlcochova, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.16.20133157\">Combined point of care nucleic acid and antibody testing for SARS-CoV-2: a prospective cohort study in suspected moderate to severe COVID-19 disease.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We assessed the diagnostic accuracy of combined rapid antibody point of care (poc) and nucleic acid assays for suspected covid-19 disease in the emergency department.<br><br>Conclusions dual point of care sars-cov-2 testing can significantly improve diagnostic sensitivity, whilst maintaining high specificity.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-14</td>\n",
       "      <td>ANNAMALAI, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.04.10.20060293\">A SIMPLE COLORIMETRIC MOLECULAR DETECTION OF NOVEL CORONAVIRUS (COVID-19), AN ESSENTIAL DIAGNOSTIC TOOL FOR PANDEMIC SCREENING</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Aim: we intended to develop and deploy a robust and rapid diagnostic methodology using lamp assay for use in point of care settings to detect sars-cov-2 infection.<br><br>Conclusion: the present study demonstrates that the lamp assay developed was a rapid, reliable, sensitive and cost effective method to detect sars-cov-2 infection in a point of care as well as in laboratory settings.a simple colorimetric molecular detection of novel coronavirus (covid-19), an essential diagnostic tool for pandemic screening.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-23</td>\n",
       "      <td>Harrington, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1128/jcm.00798-20\">Comparison of Abbott ID Now and Abbott m2000 methods for the detection of SARS-CoV-2 from nasopharyngeal and nasal swabs from symptomatic patients.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Scarborough, me) is a rapid diagnostic test that can be performed in a point of care setting equivalent to clia waived testing.….comparison of abbott id now and abbott m2000 methods for the detection of sars-cov-2 from nasopharyngeal and nasal swabs from symptomatic patients.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Wolters, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Multi-center evaluation of cepheid xpert® xpress SARS-CoV-2 point-of-care test during the SARS-CoV-2 pandemic</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>In clinical samples xpert xpress sars-cov-2 reaches an agreement of 100 % compared to all in-house rt-pcrs conclusion: cepheids genexpert xpert xpress sars-cov-2 is a valuable addition for laboratories in situations where rapid and accurate diagnostics are of the essence.multi-center evaluation of cepheid xpert® xpress sars-cov-2 point-of-care test during the sars-cov-2 pandemic.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-07-01</td>\n",
       "      <td>Li, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1101/2020.06.30.20143255\">Handyfuge-LAMP: low-cost and electricity-free centrifugation forisothermal SARS-CoV-2 detection in saliva.</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Point of care diagnostics for covid-19 detection are vital to assess infection quickly and at the source so appropriate measures can be taken.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Rubulotta, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Technologies to optimize the care of severe COVID-19 patients for healthcare providers challenged by limited resources</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The available technology has several advantages including: a) facilitating appropriate paperless documentation and communication between all healthcare givers working in isolation rooms or large isolation areas; b) testing patients and staff at the bedside using smart point of care diagnostics (spocd) to confirm covid-19 infection; c) allowing diagnostics and treatment at the bedside through point of care ultrasound (pocus) and thromboelastography (teg); d) adapting the use of anaesthetic machines and the use of volatile anaesthetics.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Tadini, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[SARS-CoV-2, a point in the storm]</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Since its emergence in december 2019, scientific knowledge about the sars-cov-2 virus has evolved rapidly but, due to the complexity and novelty of this infection and its political and economic stakes, much remains to be clarified thousands of studies have already been published and scientific research is constantly evolving in this multitude of information, we offer an update of the knowledge currently available a limitation of the propagation, the understanding of the functioning of the virus and its clinical manifestations, the administration of specific treatments, rapid and reliable diagnostic tools are the basis of the fight against this germ, which is still little known today[sars-cov-2, a point in the storm].</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-05-06</td>\n",
       "      <td>Tadini, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">[SARS-CoV-2, a point in the storm].</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>A limitation of the propagation, the understanding of the functioning of the virus and its clinical manifestations, the administration of specific treatments, rapid and reliable diagnostic tools are the basis of the fight against this germ, which is still little known today.[sars-cov-2, a point in the storm].</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-28</td>\n",
       "      <td>Inchingolo, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1016/j.ajog.2020.04.020\">The Diagnosis of Pneumonia in a Pregnant Woman with COVID-19 Using Maternal Lung Ultrasound</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>We suggest point of care lung ultrasound examination as a diagnostic imaging tool in pregnant women with suspected covid-19.the diagnosis of pneumonia in a pregnant woman with covid-19 using maternal lung ultrasound.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-04-30</td>\n",
       "      <td>Padhi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.1007/978-981-15-4814-7_9\">Laboratory Diagnosis of Novel Coronavirus Disease 2019 (COVID-19) Infection</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>The development of serological assays and point of care molecular test will further intensify the diagnostic modalities of sars-cov-2.laboratory diagnosis of novel coronavirus disease 2019 (covid-19) infection.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020-06-25</td>\n",
       "      <td>Pezzi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/10.3390/v12060686\">Development and Evaluation of a Duo SARS-CoV-2 RT-qPCR Assay Combining Two Assays Approved by the World Health Organization Targeting the Envelope and the RNA-Dependant RNA Polymerase (RdRp) Coding Regions</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Dozens of monoplex in-house rt-qpcr assays are already available; however, the development of dual-target assays is suited to avoid false-negative results caused by polymorphisms or point mutations, that can compromise the accuracy of diagnostic and screening tests.</p></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2020</td>\n",
       "      <td>Pezzi, et al.</td>\n",
       "      <td><p align=\"left\"><a href=\"https://doi.org/no data provided\">Development and Evaluation of a duo SARS-CoV-2 RT-qPCR Assay Combining Two Assays Approved by the World Health Organization Targeting the Envelope and the RNA-Dependant RNA Polymerase (RdRp) Coding Regions</a></p></td>\n",
       "      <td><p fontsize=tiny\" align=\"left\"><br><br>Dozens of monoplex in-house rt-qpcr assays are already available; however, the development of dual-target assays is suited to avoid false-negative results caused by polymorphisms or point mutations, that can compromise the accuracy of diagnostic and screening tests.</p></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  This is separate from the ipykernel package so we can avoid doing imports until\n",
      "/home/niharika/.local/lib/python3.6/site-packages/pandas/core/indexing.py:671: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self._setitem_with_indexer(indexer, value)\n",
      "/home/niharika/.local/lib/python3.6/site-packages/ipykernel_launcher.py:14: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \n"
     ]
    }
   ],
   "source": [
    "display(HTML('<h1>Task 1: What is known about transmission, incubation, and environmental stability?</h1>'))\n",
    "\n",
    "display(HTML('<h3>Results currently limited to two (10) for ease of scanning</h3>'))\n",
    "\n",
    "# list of lists of search terms\n",
    "questions=[\n",
    "['Q: What is the range of incubation periods for the disease in humans?'],\n",
    "['Q: How long are individuals are contagious?'],\n",
    "['Q: How long are individuals are contagious, even after recovery.'],\n",
    "['Q: Does the range of incubation period vary across age groups?'],\n",
    "['Q: Does the range of incubation period vary with children?'],\n",
    "['Q: Does the range of incubation period vary based on underlying health?'],\n",
    "['Q: What do we know about the basic reproduction number?'],\n",
    "['Q: What is the prevalance of asymptomatic transmission?'],\n",
    "['Q: What do we know about asymptomatic transmission in children?'],\n",
    "['Q: What do we know about seasonality of transmission?'],\n",
    "['Q: Informing decontamination based on physical science of the coronavirus?'],\n",
    "['Q: What do we know about stability of the virus in environmental conditions?'],\n",
    "['Q: What do we know about persistence of the virus on various substrates? (e.g., nasal discharge, sputum, urine, fecal matter, blood)'],\n",
    "['Q: What do we know about persistence of the virus on various surfaces? (e,g., copper, stainless steel, plastic) '],\n",
    "['Q: What do we know about viral shedding duration?'],\n",
    "['Q: What do we know about viral shedding in fecal/stool?'],\n",
    "['Q: What do we know about viral shedding from nasopharynx?'],\n",
    "['Q: What do we know about viral shedding in blood?'],\n",
    "['Q: What do we know about viral shedding in urine?'],\n",
    "['Q: What do we know about implemtation of diagnostics?'],\n",
    "['Q: What do we know about disease models?'],\n",
    "['Q: What do we know about models for disease infection?'],\n",
    "['Q: What do we know about models for disease transmission?'],\n",
    "['Q: Are there studies about phenotypic change?'],\n",
    "['Q: What is know about adaptations (mutations) of the virus?'],\n",
    "['Q: What do we know about the human immune response and immunity?'],\n",
    "['Q: Is population movement control effective in stopping transmission (spread)?'],\n",
    "['Q: Effectiveness of personal protective equipment (PPE)?'],\n",
    "['Q: What is the role of environment in transmission?']\n",
    "]\n",
    "#search=[['incubation','period','range','mean','%']]\n",
    "search=[['incubation','period','range'],\n",
    "['viral','shedding','duration'],\n",
    "['asymptomatic','shedding'],\n",
    "['incubation','period','age','statistically','significant'],\n",
    "['incubation','period','child'],\n",
    "['incubation','groups','risk'],\n",
    "['basic','reproduction', 'number','%'],\n",
    "['asymptomatic','infection','%'],\n",
    "['asymptomatic','children'],\n",
    "['seasonal','transmission'],\n",
    "['contaminat','object'],\n",
    "['environmental', 'conditions'],\n",
    "['sputum','stool','blood','urine'],\n",
    "['persistence','surfaces'],\n",
    "['duration','viral','shedding'],\n",
    "['shedding','stool'],\n",
    "['shedding','nasopharynx'],\n",
    "['shedding','blood'],\n",
    "['shedding','urine'],\n",
    "['diagnostics','point'],\n",
    "['model', 'disease'],        \n",
    "['model', 'infection'],\n",
    "['model', 'transmission'],\n",
    "['phenotypic'],\n",
    "['mutation'],\n",
    "['immune','response'],\n",
    "['restriction', 'movement'],\n",
    "['protective','clothing'],\n",
    "['transmission','routes']\n",
    "]\n",
    "\n",
    "q=0\n",
    "for search_words in search:\n",
    "    str1=''\n",
    "    # a make a string of the search words to print readable version above table\n",
    "    str1=' '.join(questions[q])\n",
    "    \n",
    "    #search the dataframe for all words\n",
    "    df1=search_dataframe(df,search_words)\n",
    "\n",
    "    # analyze search results for relevance \n",
    "    df1=search_relevance(df1,search_words)\n",
    "\n",
    "    # get best sentences\n",
    "    df_table=get_sentences(df1,search_words)\n",
    "    \n",
    "    length=df_table.shape[0]\n",
    "    #limit 3 results\n",
    "    df_table=df_table.head(15)\n",
    "    df_table=df_table.drop(['rel_score'], axis=1)\n",
    "    #convert df to html\n",
    "    df_table=HTML(df_table.to_html(escape=False,index=False))\n",
    "    \n",
    "    # display search topic\n",
    "    display(HTML('<h3>'+str1+'</h3>'))\n",
    "    \n",
    "    #display table\n",
    "    if length<1:\n",
    "        print (\"No reliable answer could be located in the literature\")\n",
    "    else:\n",
    "        display(df_table)\n",
    "    q=q+1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
